<!DOCTYPE html>
<html class="no-js" lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<title>Bioinformatics-Links - DNA Academic</title>
	<script>(function(d,e){d[e]=d[e].replace("no-js","js");})(document.documentElement,"className");</script>
	<meta name="description" content="Bioinformatics-Links">
		<meta property="og:title" content="Bioinformatics-Links" />
<meta property="og:description" content="Bioinformatics-Links" />
<meta property="og:type" content="article" />
<meta property="og:url" content="/bioinformatics-links/" />
<meta property="article:published_time" content="2022-08-21T00:00:00&#43;00:00"/>
<meta property="article:modified_time" content="2022-08-21T00:00:00&#43;00:00"/>

		
<meta itemprop="name" content="Bioinformatics-Links">
<meta itemprop="description" content="Bioinformatics-Links">


<meta itemprop="datePublished" content="2022-08-21T00:00:00&#43;00:00" />
<meta itemprop="dateModified" content="2022-08-21T00:00:00&#43;00:00" />
<meta itemprop="wordCount" content="8078">



<meta itemprop="keywords" content="Bi,Linux," />

		<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Bioinformatics-Links"/>
<meta name="twitter:description" content="Bioinformatics-Links"/>

	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
	<link rel="dns-prefetch" href="//fonts.googleapis.com">
	<link rel="dns-prefetch" href="//fonts.gstatic.com">
	<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:400,400i,700">

	<link rel="stylesheet" href="/css/style.css">
	<link rel="stylesheet" href="/css/custom.css">

	<link rel="shortcut icon" href="/favicon.ico">
		
</head>
<body class="body">
	<div class="container container--outer">
		<header class="header">
	<div class="container header__container">
		
	<div class="logo logo--mixed">
		<a class="logo__link" href="/" title="DNA Academic" rel="home">
			<div class="logo__item logo__imagebox">
					<img class="logo__img" src="/img/logo.svg">
				</div><div class="logo__item logo__text">
					<div class="logo__title">DNA Academic</div>
					<div class="logo__tagline">dna.ac.cn</div>
				</div>
		</a>
	</div>
		
<nav class="menu">
	<button class="menu__btn" aria-haspopup="true" aria-expanded="false" tabindex="0">
		<span class="menu__btn-title" tabindex="-1">Menu</span>
	</button>
	<ul class="menu__list">
		<li class="menu__item menu__item--active">
			<a class="menu__link" href="/bioinformatics-links/">
				
				<span class="menu__text">Bioinformatics-Links</span>
				
			</a>
		</li>
		<li class="menu__item">
			<a class="menu__link" href="/about/">
				
				<span class="menu__text">Example article title</span>
				
			</a>
		</li>
	</ul>
</nav>

	</div>
</header>
		<div class="wrapper flex">
			<div class="primary">
			
<main class="main" role="main">
	<article class="post">
		<header class="post__header">
			<h1 class="post__title">Bioinformatics-Links</h1>
			<p class="post__lead">Example lead - highlighted near the title</p>
			<div class="post__meta meta"><div class="meta__item-author meta__item">
	<svg class="meta__icon icon icon-author" width="16" height="16" viewBox="0 0 12 16"><path d="M6 1c2.2 0 3.5 2 3.5 4.5C9.5 7 8.9 8.2 8 9c2.9.8 4 2.5 4 5v1H0v-1c0-2.5 1.1-4.2 4-5-.9-.8-1.5-2-1.5-3.5C2.5 3 3.8 1 6 1z"/></svg><span class="meta__text">Ricky Wang</span>
</div>
<div class="meta__item-datetime meta__item">
	<svg class="meta__icon icon icon-time" width="16" height="14" viewBox="0 0 30 28"><path d="M15 0C7 0 1 6 1 14s6 14 14 14 14-6 14-14S23 0 15 0zm0 25C9 25 4 20 4 14S9 3 15 3s11 5 11 11-5 11-11 11zm1-18h-2v8.4l6.8 4.4L22 18l-6-3.8V7z"/></svg><time class="meta__text" datetime="2022-08-21T00:00:00Z">2022-08-21</time></div><div class="meta__item-categories meta__item"><svg class="meta__icon icon icon-category" width="16" height="16" viewBox="0 0 16 16"><path d="m7 2l1 2h8v11h-16v-13z"/></svg><span class="meta__text"><a class="meta__link" href="/categories/bioinformatics/" rel="category">Bioinformatics</a>, <a class="meta__link" href="/categories/linux/" rel="category">Linux</a>
	</span>
</div></div>
		</header>
		
	<figure class="post__thumbnail thumbnail">
		
		<img class="thumbnail__image" src="/img/placeholder.png" alt="Bioinformatics-Links">
		
	</figure>
<div class="post__toc toc">
	<div class="toc__title">Page content</div>
	<div class="toc__menu">
		<nav id="TableOfContents">
<ul>
<li>
<ul>
<li>
<ul>
<li><a href="#national-genomics-data-center-https-bigd-big-ac-cn-lang-en-国家基因组科学数据中心"><a href="https://bigd.big.ac.cn/?lang=en">National Genomics Data Center</a>  国家基因组科学数据中心</a></li>
<li><a href="#nucleotide-sequence-databases-核酸序列数据库">Nucleotide Sequence Databases  核酸序列数据库</a></li>
<li><a href="#protein-sequence-databases-蛋白质序列数据库">Protein Sequence Databases  蛋白质序列数据库</a></li>
<li><a href="#database-searching-by-sequence-similarity-序列相似性搜索">Database Searching by Sequence Similarity  序列相似性搜索</a></li>
<li><a href="#sequence-alignment-序列比对">Sequence Alignment  序列比对</a></li>
<li><a href="#human-genome-databases-人类基因组数据库">Human Genome Databases  人类基因组数据库</a></li>
<li><a href="#next-generation-sequencing-下一代测序">Next Generation Sequencing  下一代测序</a></li>
<li><a href="#rice-databases-水稻数据库">Rice databases  水稻数据库</a></li>
<li><a href="#plant-promoter-and-regulatory-element-resources-植物启动子和调控元件资源">Plant Promoter and Regulatory Element Resources  植物启动子和调控元件资源</a></li>
<li><a href="#databases-of-other-organisms-其他物种数据库">Databases of other Organisms  其他物种数据库</a></li>
<li><a href="#genome-wide-analysis-基因组分析">Genome-wide Analysis  基因组分析</a></li>
<li><a href="#protein-properties-蛋白质理化参数">Protein Properties  蛋白质理化参数</a></li>
<li><a href="#protein-domains-蛋白质结构域">Protein Domains  蛋白质结构域</a></li>
<li><a href="#motif-and-pattern-search-in-sequences-序列模体和样式搜索">Motif and Pattern Search in Sequences  序列模体和样式搜索</a></li>
<li><a href="#protein-3d-structure-蛋白质三维结构">Protein 3D Structure  蛋白质三维结构</a></li>
<li><a href="#phylogeny-taxonomy-进化与分类">Phylogeny &amp; Taxonomy  进化与分类</a></li>
<li><a href="#gene-prediction-基因预测">Gene Prediction  基因预测</a></li>
<li><a href="#gene-structure-analysis-基因结构分析">Gene Structure Analysis  基因结构分析</a></li>
<li><a href="#gene-expression-databases-基因表达数据库">Gene Expression Databases  基因表达数据库</a></li>
<li><a href="#gene-regulation-基因调控">Gene Regulation  基因调控</a></li>
<li><a href="#small-rna-microrna-小分子rna-微rna">Small RNA/MicroRNA  小分子RNA/微RNA</a></li>
<li><a href="#metabolic-gene-regulatory-signal-transduction-network-databases-代谢-基因调控和信号转导网络数据库">Metabolic, Gene Regulatory &amp; Signal Transduction Network Databases  代谢、基因调控和信号转导网络数据库</a></li>
<li><a href="#systems-biology-系统生物学">Systems Biology  系统生物学</a></li>
<li><a href="#synthetic-biology-合成生物学">Synthetic Biology  合成生物学</a></li>
<li><a href="#other-databases-annotations-ontologies-consortia-etc">Other Databases (Annotations, Ontologies, Consortia, etc.)</a></li>
<li><a href="#miscellaneous-tools-其他工具">Miscellaneous Tools  其他工具</a></li>
<li><a href="#computational-resources-计算资源">Computational Resources  计算资源</a></li>
<li><a href="#human-diseases-and-cancer-databases-人类疾病与癌症数据库">Human Diseases and Cancer Databases  人类疾病与癌症数据库</a></li>
<li><a href="#medical-image-repositorie-医学影像库">Medical Image Repositorie  医学影像库</a></li>
<li><a href="#bioinformatics-on-line-course-materials-and-tutorials-生物信息学在线课程教材">Bioinformatics on-line course materials and tutorials  生物信息学在线课程教材</a></li>
<li><a href="#background-information-news-背景信息与新闻">Background Information &amp; News  背景信息与新闻</a></li>
<li><a href="#bioinformatics-conferences-国际会议">Bioinformatics Conferences  国际会议</a></li>
<li><a href="#other-collections-其他资源">Other Collections  其他资源</a></li>
<li><a href="#bioinformatics-journals-相关期刊">Bioinformatics Journals  相关期刊</a></li>
</ul></li>
</ul></li>
</ul>
</nav>
	</div>
</div><div class="content post__content clearfix">
			

<h3 id="national-genomics-data-center-https-bigd-big-ac-cn-lang-en-国家基因组科学数据中心"><a href="https://bigd.big.ac.cn/?lang=en">National Genomics Data Center</a>  国家基因组科学数据中心</h3>

<ul>
<li><h3 id="nucleotide-sequence-databases-核酸序列数据库">Nucleotide Sequence Databases  核酸序列数据库</h3>

<ul>
<li><a href="http://www.ncbi.nlm.nih.gov/">NCBI</a><br />

<ul>
<li>National Center for Biotechnology Information</li>
</ul></li>
<li><a href="https://www.ebi.ac.uk/services">EBI</a><br />

<ul>
<li>European Bioinformatics Institute</li>
</ul></li>
<li><a href="http://www.ddbj.nig.ac.jp/">DDBJ</a><br />

<ul>
<li>DNA Data Bank of Japan</li>
</ul></li>
</ul></li>

<li><h3 id="protein-sequence-databases-蛋白质序列数据库">Protein Sequence Databases  蛋白质序列数据库</h3>

<ul>
<li><a href="http://www.expasy.ch/sprot/sprot-top.html">SWISS-PROT &amp; TrEMBL</a><br />

<ul>
<li>Protein sequence database and computer annotated supplement</li>
</ul></li>
<li><a href="http://www.ebi.uniprot.org/" title="UniProt">UniProt</a><br />

<ul>
<li>(Universal Protein Resource) is the world&rsquo;s most comprehensive catalog of information on proteins. It is a central repository of protein sequence and function created by joining the information contained in Swiss-Prot, TrEMBL, and PIR.</li>
</ul></li>
<li><a href="http://pir.georgetown.edu/pirwww/">PIR</a><br />

<ul>
<li>Protein Information Resource</li>
</ul></li>
<li><a href="http://www.hupo.org/" title="HUPO">HUPO</a><br />

<ul>
<li>HUman Proteome Organization</li>
</ul></li>
</ul></li>

<li><h3 id="database-searching-by-sequence-similarity-序列相似性搜索">Database Searching by Sequence Similarity  序列相似性搜索</h3>

<ul>
<li><a href="http://www.ncbi.nlm.nih.gov/BLAST/">BLAST @ NCBI</a></li>
<li><a href="http://www.ncbi.nlm.nih.gov/blast/Blast.cgi?ALIGNMENTS=250&amp;ALIGNMENT_VIEW=Pairwise&amp;CLIENT=web&amp;COMPOSITION_BASED_STATISTICS=on&amp;DATABASE=nr&amp;CDD_SEARCH=on&amp;DESCRIPTIONS=500&amp;ENTREZ_QUERY=(none)&amp;EXPECT=10&amp;FORMAT_OBJECT=Alignment&amp;FORMAT_TYPE=HTML&amp;HITLIST_SIZE=500&amp;I_THRESH=0.002&amp;MATRIX_NAME=BLOSUM62&amp;NCBI_GI=on&amp;PAGE=Proteins&amp;PROGRAM=blastp&amp;RUN_PSIBLAST=on&amp;SERVICE=plain&amp;SET_DEFAULTS.x=36&amp;SET_DEFAULTS.y=5&amp;SHOW_OVERVIEW=on&amp;AUTO_FORMAT=yes&amp;END_OF_HTTPGET=Yes">PSI-BLAST @ NCBI</a></li>
<li><a href="http://www.ebi.ac.uk/Tools/sss/fasta/">FASTA @ EBI</a></li>
<li><a href="http://genome.ucsc.edu/">BLAT</a><br />
Jim Kent&rsquo;s Blat is just superb in terms of speed and the integrated view you get for viewing the results</li>
</ul></li>

<li><h3 id="sequence-alignment-序列比对">Sequence Alignment  序列比对</h3>

<ul>
<li><a href="http://tcoffee.crg.cat/apps/tcoffee/index.html">T-COFFEE</a><br />

<ul>
<li>multiple sequence alignment</li>
</ul></li>
<li><a href="https://www.ebi.ac.uk/Tools/msa/clustalo/">Clustal Omega @ EBI</a><br />

<ul>
<li>multiple sequence alignment</li>
</ul></li>
<li><a href="http://bioweb.pasteur.fr/seqanal/interfaces/boxshade.html">BOXSHADE</a><br />

<ul>
<li>pretty printing and shading of multiple alignments</li>
</ul></li>
<li><a href="http://www.ncbi.nlm.nih.gov/sutils/splign/" title="Splign">Splign</a><br />

<ul>
<li>Splign is a utility for computing cDNA-to-Genomic, or spliced sequence alignments. At the heart of the program is a global alignment algorithm that specifically accounts for introns and splice signals.</li>
</ul></li>
<li><a href="http://www.ncbi.nlm.nih.gov/IEB/Research/Ostell/Spidey/index.html">Spidey</a><br />

<ul>
<li>an mRNA-to-genomic alignment program</li>
</ul></li>
<li><a href="http://pbil.univ-lyon1.fr/sim4.php">SIM4</a><br />

<ul>
<li>a program to align cDNA and genomic DNA</li>
</ul></li>
<li><a href="http://bio.cse.psu.edu/pipmaker">PipMaker</a><br />

<ul>
<li>computes alignments of similar regions in two (long) DNA sequences</li>
</ul></li>
<li><a href="http://www-gsd.lbl.gov/VISTA/">VISTA</a><br />

<ul>
<li>align + detect conserved regions in long genomic sequences</li>
</ul></li>
</ul></li>

<li><h3 id="human-genome-databases-人类基因组数据库">Human Genome Databases  人类基因组数据库</h3>

<ul>
<li><a href="http://www.ncbi.nlm.nih.gov/genome/guide/human/">Draft Human Genome @ NCBI</a></li>
<li><a href="http://genome.ucsc.edu/">Draft Human Genome @ UCSC</a></li>
<li><a href="http://www.ensembl.org/">Ensembl</a><br />

<ul>
<li>automatically annotated human genome.</li>
</ul></li>
<li><a href="http://gdb.org/">GDB</a><br />

<ul>
<li>Genome Database</li>
</ul></li>
<li><a href="http://mgc.nci.nih.gov/">Mammalian Gene Collection</a><br />

<ul>
<li>full-length (open reading frame) sequences for human and mouse</li>
</ul></li>
<li><a href="http://www.genecards.org/">GeneCards</a><br />

<ul>
<li>human genes, proteins and diseases
<br />
<br />
<br /></li>
</ul></li>
</ul></li>

<li><h3 id="next-generation-sequencing-下一代测序">Next Generation Sequencing  下一代测序</h3>

<ul>
<li><p><a href="http://www.oxfordjournals.org/our_journals/bioinformatics/nextgenerationsequencing.html">Bioinformatics Sequencing</a> Journal for Next Generation</p></li>

<li><p> <strong>Integrated solutions  集成解决方案</strong></p></li>
</ul>

<p><a href="http://www.clcbio.com/index.php?id=1240">CLCbio Genomics Workbench</a><br />
- <em>de novo</em> and reference assembly of Sanger, Roche FLX, Illumina, Helicos, and SOLiD data. Commercial next-gen-seq software that extends the CLCbio Main Workbench software. Includes SNP detection, CHiP-seq, browser and other features. Commercial. Windows, Mac OS X and Linux.<br />
<a href="http://g2.trac.bx.psu.edu/">Galaxy</a><br />
- Galaxy = interactive and reproducible genomics. A job webportal.<br />
<a href="http://www.genomatix.de/index.html">Genomatix</a><br />
- Integrated Solutions for Next Generation Sequencing data analysis.<br />
<a href="http://www.jmp.com/software/genomics/">JMP Genomics</a><br />
- Next gen visualization and statistics tool from SAS. They are working with <a href="http://www.marketwatch.com/news/story/JMPR-Genomics-NCGR-Partnership-Foster/story.aspx?guid=%7B7AC9DE36-B6AA-4EDE-9CD5-633B29FE6154%7D">NCGR</a> to refine this tool and produce others.<br />
<a href="http://softgenetics.com/NextGENe.html">NextGENe</a><br />
- <em>de novo</em> and reference assembly of Illumina, SOLiD and Roche FLX data. Uses a novel Condensation Assembly Tool approach where reads are joined via &ldquo;anchors&rdquo; into mini-contigs before assembly. Includes SNP detection, CHiP-seq, browser and other features. Commercial. Win or MacOS.<br />
<a href="http://www.dnastar.com/products/SMGA.php">SeqMan Genome Analyser</a><br />
- Software for Next Generation sequence assembly of Illumina, Roche FLX and Sanger data integrating with Lasergene Sequence Analysis software for additional analysis and visualization capabilities. Can use a hybrid templated/de novo approach. Commercial. Win or Mac OS X.<br />
<a href="http://1001genomes.org/software/shore.html">SHORE</a><br />
- SHORE, for Short Read, is a mapping and analysis pipeline for short DNA sequences produced on a Illumina Genome Analyzer. A suite created by the 1001 Genomes project. Source for POSIX.<br />
<a href="http://www.realtimegenomics.com/">SlimSearch</a><br />
- Fledgling commercial product.</p>

<ul>
<li><strong>Align/Assemble to a reference  参考序列比对/组装</strong>
<br /></li>
</ul>

<p><a href="http://sourceforge.net/apps/mediawiki/bfast/index.php?title=Main_Page">BFAST</a><br />
- Blat-like Fast Accurate Search Tool. Written by Nils Homer, Stanley F. Nelson and Barry Merriman at UCLA.<br />
<a href="http://bowtie-bio.sourceforge.net/">Bowtie</a><br />
- Ultrafast, memory-efficient short read aligner. It aligns short DNA sequences (reads) to the human genome at a rate of 25 million reads per hour on a typical workstation with 2 gigabytes of memory. Uses a Burrows-Wheeler-Transformed (BWT) index. <a href="http://seqanswers.com/forums/showthread.php?t=706">Link to discussion thread here</a>. Written by Ben Langmead and Cole Trapnell. Linux, Windows, and Mac OS X.<br />
<a href="http://www.ebi.ac.uk/~guy/exonerate/">Exonerate</a> - Various forms of pairwise alignment (including Smith-Waterman-Gotoh) of DNA/protein against a reference. Authors are Guy St C Slater and Ewan Birney from EMBL. C for POSIX.<br />
<a href="http://1001genomes.org/software/genomemapper.html">GenomeMapper</a><br />
- GenomeMapper is a short read mapping tool designed for accurate read alignments. It quickly aligns millions of reads either with ungapped or gapped alignments. A tool created by the 1001 Genomes project. Source for POSIX.<br />
<a href="http://research-pub.gene.com/gmap/">GMAP</a><br />
- GMAP (Genomic Mapping and Alignment Program) for mRNA and EST Sequences. Developed by Thomas Wu and Colin Watanabe at Genentec. C/Perl for Unix.<br />
<a href="http://dna.cs.byu.edu/gnumap/">gnumap</a><br />
- The Genomic Next-generation Universal MAPper (gnumap) is a program designed to accurately map sequence data obtained from next-generation sequencing machines (specifically that of Solexa/Illumina) back to a genome of any size. It seeks to align reads from nonunique repeats using statistics. From authors at Brigham Young University. C source/Unix.<br />
<a href="http://sourceforge.net/projects/maq/">MAQ</a> - Mapping and Assembly with Qualities (renamed from MAPASS2). Particularly designed for Illumina with preliminary functions to handle ABI SOLiD data. Written by Heng Li from the Sanger Centre. Features extensive supporting tools for DIP/SNP detection, etc. C++ source<br />
<a href="http://bioinformatics.bc.edu/marthlab/wiki/index.php/Software">MOSAIK</a> - MOSAIK produces gapped alignments using the Smith-Waterman algorithm. Features a number of support tools. Support for Roche FLX, Illumina, SOLiD, and Helicos. Written by Michael Strömberg at Boston College. Win/Linux/MacOSX<br />
<a href="http://www.novocraft.com/index.html">Novocraft</a><br />
- Tools for reference alignment of paired-end and single-end Illumina reads. Uses a Needleman-Wunsch algorithm. Can support Bis-Seq. Commercial. Available free for evaluation, educational use and for use on open not-for-profit projects. Requires Linux or Mac OS X.<br />
<a href="http://pass.cribi.unipd.it/cgi-bin/pass.pl">PASS</a><br />
- It supports Illumina, SOLiD and Roche-FLX data formats and allows the user to modulate very finely the sensitivity of the alignments. Spaced seed intial filter, then NW dynamic algorithm to a SW(like) local alignment. Authors are from CRIBI in Italy. Win/Linux.<br />
<a href="http://rulai.cshl.edu/rmap/">RMAP</a><br />
- Assembles 20 - 64 bp Illumina reads to a FASTA reference genome. By Andrew D. Smith and Zhenyu Xuan at CSHL. (published in BMC Bioinformatics). POSIX OS required.<br />
<a href="http://compbio.cs.toronto.edu/shrimp/">SHRiMP</a><br />
- Assembles to a reference sequence. Developed with Applied Biosystem&rsquo;s colourspace genomic representation in mind. Authors are Michael Brudno and Stephen Rumble at the University of Toronto. POSIX.<br />
<a href="http://www.bcgsc.ca/platform/bioinfo/software/slider">Slider</a><br />
- An application for the Illumina Sequence Analyzer output that uses the probability files instead of the sequence files as an input for alignment to a reference sequence or a set of reference sequences. Authors are from BCGSC. Paper is <a href="http://seqanswers.com/forums/showthread.php?t=740">here</a>.<br />
<a href="http://soap.genomics.org.cn/">SOAP</a><br />
(Short Oligonucleotide Alignment Program). A program for efficient gapped and ungapped alignment of short oligonucleotides onto reference sequences. The updated version uses a BWT. Can call SNPs and INDELs. Author is Ruiqiang Li at the Beijing Genomics Institute. C++, POSIX.<br />
<a href="http://www.sanger.ac.uk/Software/analysis/SSAHA/">SSAHA2</a><br />
-(Sequence Search and Alignment by Hashing Algorithm) is a tool for rapidly finding near exact matches in DNA or protein databases using a hash table. Developed at the Sanger Centre by Zemin Ning, Anthony Cox and James Mullikin. C++ for Linux/Alpha.<br />
<a href="http://socs.biology.gatech.edu/">SOCS</a><br />
- Aligns SOLiD data. SOCS is built on an iterative variation of the Rabin-Karp string search algorithm, which uses hashing to reduce the set of possible matches, drastically increasing search speed. Authors are Ondov B, Varadarajan A, Passalacqua KD and Bergman NH.<br />
<a href="http://bibiserv.techfak.uni-bielefeld.de/swift/welcome.html">SWIFT</a><br />
- The SWIFT suit is a software collection for fast index-based sequence comparison. It contains: SWIFT — fast local alignment search, guaranteeing to find epsilon-matches between two sequences. SWIFT BALSAM — a very fast program to find semiglobal non-gapped alignments based on k-mer seeds. Authors are Kim Rasmussen (SWIFT) and Wolfgang Gerlach (SWIFT BALSAM)<br />
<a href="http://www.vmatch.de/">Vmatch</a><br />
- A versatile software tool for efficiently solving large scale sequence matching tasks. Vmatch subsumes the software tool REPuter, but is much more general, with a very flexible user interface, and improved space and time requirements. Essentially a large string matching toolbox.</p>

<ul>
<li><strong><em>De novo</em> Align/Assemble  从头比对/组装</strong>
<br /></li>
</ul>

<p><a href="http://www.bcgsc.ca/platform/bioinfo/software/abyss">ABySS</a><br />
- Assembly By Short Sequences. ABySS is a de novo sequence assembler that is designed for very short reads. The single-processor version is useful for assembling genomes up to 40-50 Mbases in size. The parallel version is implemented using MPI and is capable of assembling larger genomes. By Simpson JT and others at the Canada&rsquo;s Michael Smith Genome Sciences Centre. C++ as source.<br />
<a href="http://www.broad.mit.edu/science/programs/genome-biology/computational-rd/computational-research-and-development">ALLPATHS</a><br />
- ALLPATHS: De novo assembly of whole-genome shotgun microreads. ALLPATHS is a whole genome shotgun assembler that can generate high quality assemblies from short reads. Assemblies are presented in a graph form that retains ambiguities, such as those arising from polymorphism, thereby providing information that has been absent from previous genome assemblies. Broad Institute.<br />
<a href="http://www.genomic.ch/edena.php">Edena</a><br />
- Edena (Exact DE Novo Assembler) is an assembler dedicated to process the millions of very short reads produced by the Illumina Genome Analyzer. Edena is based on the traditional overlap layout paradigm. By D. Hernandez, P. François, L. Farinelli, M. Osteras, and J. Schrenzel. Linux/Win.<br />
[](<a href="http://euler-assembler.ucsd.edu/portal/)[SEQAN](http://www.seqan.de/">http://euler-assembler.ucsd.edu/portal/)[SEQAN](http://www.seqan.de/</a>)<br />
- A Consistency-based Consensus Algorithm for De Novo and Reference-guided Sequence Assembly of Short Reads. By Tobias Rausch and others. C++, Linux/Win.<br />
<a href="http://sharcgs.molgen.mpg.de/">SHARCGS</a><br />
- De novo assembly of short reads. Authors are Dohm JC, Lottaz C, Borodina T and Himmelbauer H. from the Max-Planck-Institute for Molecular Genetics.<br />
<a href="http://www.bcgsc.ca/platform/bioinfo/software/ssake">SSAKE</a> - The Short Sequence Assembly by K-mer search and 3&rsquo; read Extension (SSAKE) is a genomics application for aggressively assembling millions of short nucleotide sequences by progressively searching for perfect 3&rsquo;-most k-mers using a DNA prefix tree. Authors are René Warren, Granger Sutton, Steven Jones and Robert Holt from the Canada&rsquo;s Michael Smith Genome Sciences Centre. Perl/Linux.<br />
<a href="http://soap.genomics.org.cn/">SOAPdenovo</a><br />
- Part of the SOAP suite. See above.<br />
<a href="https://sourceforge.net/projects/vcake">VCAKE</a><br />
- De novo assembly of short reads with robust error correction. An improvement on early versions of SSAKE.<br />
<a href="http://www.ebi.ac.uk/~zerbino/velvet/">Velvet</a><br />
- a de novo genomic assembler specially designed for short read sequencing technologies, such as Solexa or 454. Need about 20-25X coverage and paired reads. Developed by Daniel Zerbino and Ewan Birney at the European Bioinformatics Institute (EMBL-EBI).</p>

<ul>
<li><strong>SNP/Indel Discovery  单核苷酸多态性位点/插入缺失位点寻找</strong>
<br /></li>
</ul>

<p><a href="http://www.sanger.ac.uk/Software/analysis/ssahaSNP/">ssahaSNP</a><br />
- ssahaSNP is a polymorphism detection tool. It detects homozygous SNPs and indels by aligning shotgun reads to the finished genome sequence. Highly repetitive elements are filtered out by ignoring those kmer words with high occurrence numbers. More tuned for ABI Sanger reads. Developers are Adam Spargo and Zemin Ning from the Sanger Centre. Compaq Alpha, Linux-64, Linux-32, Solaris and Mac<br />
<a href="http://bioinformatics.bc.edu/marthlab/wiki/index.php/PyroBayes">PyroBayes</a><br />
- PyroBayes is a novel base caller for pyrosequences from the 454 Life Sciences sequencing machines. It was designed to assign more accurate base quality estimates to the 454 pyrosequences. Developers at Boston College.</p>

<ul>
<li><strong>Genome Annotation/Genome Browser/Alignment Viewer/Assembly Database  基因组注释/基因组浏览器/比对浏览器/装配数据库</strong>
<br /></li>
</ul>

<p><a href="http://bioinformatics.bc.edu/marthlab/wiki/index.php/EagleView">EagleView</a><br />
- An information-rich genome assembler viewer. EagleView can display a dozen different types of information including base quality and flowgram signal. Developers at Boston College.<br />
<a href="http://www.sanger.ac.uk/Software/analysis/lookseq/">LookSeq</a><br />
- LookSeq is a web-based application for alignment visualization, browsing and analysis of genome sequence data. LookSeq supports multiple sequencing technologies, alignment sources, and viewing modes; low or high-depth read pileups; and easy visualization of putative single nucleotide and structural variation. From the Sanger Centre.<br />
<a href="http://evolution.sysu.edu.cn/mapview/">MapView</a> - MapView: visualization of short reads alignment on desktop computer. From the Evolutionary Genomics Lab at Sun-Yat Sen University, China. Linux.<br />
<a href="http://www.bcgsc.ca/platform/bioinfo/software/sam">SAM</a> - Sequence Assembly Manager. Whole Genome Assembly (WGA) Management and Visualization Tool. It provides a generic platform for manipulating, analyzing and viewing WGA data, regardless of input type. Developers are Rene Warren, Yaron Butterfield, Asim Siddiqui and Steven Jones at Canada&rsquo;s Michael Smith Genome Sciences Centre. MySQL backend and Perl-CGI web-based frontend/Linux.<br />
<a href="http://sourceforge.net/projects/staden/">STADEN</a><br />
- Includes GAP4. GAP5 once completed will handle next-gen sequencing data. A partially implemented test version is available <a href="https://sourceforge.net/project/show...kage_id=256957">here</a><br />
<a href="http://www.bcgsc.ca/platform/bioinfo/software/xmatchview">XMatchView</a><br />
- A visual tool for analyzing cross_match alignments. Developed by Rene Warren and Steven Jones at Canada&rsquo;s Michael Smith Genome Sciences Centre. Python/Win or Linux.</p>

<ul>
<li><strong>Counting  计数</strong>
<br /></li>
</ul>

<p><a href="http://epigenomics.mcdb.ucla.edu/BS-Seq/download.html">BS-Seq</a><br />
- The source code and data for the &ldquo;Shotgun Bisulphite Sequencing of the Arabidopsis Genome Reveals DNA Methylation Patterning&rdquo; Nature paper by <a href="http://www.ncbi.nlm.nih.gov/sites/entrez?holding=&amp;db=pubmed&amp;cmd=search&amp;term=Shotgun%20Bisulphite%20Sequencing">Cokus et al.</a> (Steve Jacobsen&rsquo;s lab at UCLA). POSIX.<br />
<a href="http://woldlab.caltech.edu/">CHiPSeq</a><br />
- Program used by Johnson et al. (2007) in their Science publication<br />
<a href="http://tiger.dbs.nus.edu.sg/cnv-seq/">CNV-Seq</a><br />
- CNV-seq, a new method to detect copy number variation using high-throughput sequencing. Chao Xie and Martti T Tammi at the National University of Singapore. Perl/R.<br />
* <a href="http://www.bcgsc.ca/platform/bioinfo/software/findpeaks">FindPeaks</a><br />
- perform analysis of ChIP-Seq experiments. It uses a naive algorithm for identifying regions of high coverage, which represent Chromatin Immunoprecipitation enrichment of sequence fragments, indicating the location of a bound protein of interest. Original algorithm by Matthew Bainbridge, in collaboration with Gordon Robertson. Current code and implementation by Anthony Fejes. Authors are from the Canada&rsquo;s Michael Smith Genome Sciences Centre. JAVA/OS independent. Latest versions available as part of the <a href="http://vancouvershortr.sourceforge.net/">Vancouver Short Read Analysis Package</a><br />
<a href="http://liulab.dfci.harvard.edu/MACS/">MACS</a><br />
- Model-based Analysis for ChIP-Seq. MACS empirically models the length of the sequenced ChIP fragments, which tends to be shorter than sonication or library construction size estimates, and uses it to improve the spatial resolution of predicted binding sites. MACS also uses a dynamic Poisson distribution to effectively capture local biases in the genome sequence, allowing for more sensitive and robust prediction. Written by Yong Zhang and Tao Liu from Xiaole Shirley Liu&rsquo;s Lab.<br />
<a href="http://www.gersteinlab.org/proj/PeakSeq/">PeakSeq</a><br />
- PeakSeq: Systematic Scoring of ChIP-Seq Experiments Relative to Controls. a two-pass approach for scoring ChIP-Seq data relative to controls. The first pass identifies putative binding sites and compensates for variation in the mappability of sequences across the genome. The second pass filters out sites that are not significantly enriched compared to the normalized input DNA and computes a precise enrichment and significance. By Rozowsky J et al. C/Perl.<br />
<a href="http://mendel.stanford.edu/sidowlab/downloads/quest/">QuEST</a> - Quantitative Enrichment of Sequence Tags. Sidow and Myers Labs at Stanford. From the 2008 publication <a href="http://www.ncbi.nlm.nih.gov/pubmed/18711362">Genome-wide analysis of transcription factor binding sites based on ChIP-Seq data</a>. (C++)<br />
<a href="http://dir.nhlbi.nih.gov/papers/lmi/epigenomes/sissrs/">SISSRs</a><br />
- Site Identification from Short Sequence Reads. BED file input. Raja Jothi @ NIH. Perl.<br />
See also <a href="http://seqanswers.com/forums/showthread.php?t=742">this thread</a> for ChIP-Seq.</p>

<ul>
<li><strong>Alternate Base Calling  可替换碱基识别</strong>
<br /></li>
</ul>

<p><a href="http://svitsrv25.epfl.ch/R-doc/library/Rolexa/html/00Index.html">Rolexa</a><br />
- R-based framework for base calling of Solexa data. Project <a href="http://www.biomedcentral.com/1471-2105/9/431">publication</a><br />
<a href="http://hannonlab.cshl.edu/Alta-Cyclic/main.html">Alta-cyclic</a><br />
- &ldquo;a novel Illumina Genome-Analyzer (Solexa) base caller&rdquo;</p>

<ul>
<li><strong>Transcriptomics  转录组学</strong>
<br /></li>
</ul>

<p><a href="http://woldlab.caltech.edu/rnaseq/">ERANGE</a><br />
- Mapping and Quantifying Mammalian Transcriptomes by RNA-Seq. Supports Bowtie, BLAT and ELAND. From the Wold lab.<br />
<a href="http://www.genoscope.cns.fr/externe/gmorse/">G-Mo.R-Se</a><br />
- G-Mo.R-Se is a method aimed at using RNA-Seq short reads to build de novo gene models. First, candidate exons are built directly from the positions of the reads mapped on the genome (without any ab initio assembly of the reads), and all the possible splice junctions between those exons are tested against unmapped reads. From CNS in France.<br />
<a href="http://evolution.sysu.edu.cn/english/software/mapnext.htm">MapNext</a><br />
- A software tool for spliced and unspliced alignments and SNP detection of short sequence reads. From the Evolutionary Genomics Lab at Sun-Yat Sen University, China.<br />
<a href="http://www.fml.tuebingen.mpg.de/raetsch/suppl/qpalma">QPalma</a><br />
- Optimal Spliced Alignments of Short Sequence Reads. Authors are Fabio De Bona, Stephan Ossowski, Korbinian Schneeberger, and Gunnar Rätsch. A paper is <a href="http://www.fml.tuebingen.mpg.de/raetsch/suppl/qpalma/qpalma-final.pdf">available</a>.<br />
<a href="http://tophat.cbcb.umd.edu/">TopHat</a><br />
- a fast splice junction mapper for RNA-Seq reads. It aligns RNA-Seq reads to mammalian-sized genomes using the ultra high-throughput short read aligner Bowtie, and then analyzes the mapping results to identify splice junctions between exons. TopHat is a collaborative effort between the University of Maryland and the University of California, Berkeley</p></li>

<li><h3 id="rice-databases-水稻数据库">Rice databases  水稻数据库</h3>

<ul>
<li><a href="http://rise.genomics.org.cn/">BGI-RISe</a><br />

<ul>
<li>Beijing Genomics Institute Rice Information System</li>
</ul></li>
<li><a href="http://rgp.dna.affrc.go.jp/">RGP</a><br />

<ul>
<li>Rice Genome Research Program</li>
</ul></li>
<li><a href="http://bis.zju.edu.cn/ricenetdb/">RiceNetDB</a><br />

<ul>
<li>is currently the most comprehensive regulatory database on Oryza sativa based on genome annotation. It was displayed in three levels: GEM, PPIs and GRNs to facilitate biomolecular regulatory analysis and gene-metabolite mapping.</li>
</ul></li>
<li><a href="http://mips.gsf.de/proj/plant/jsf/rice/index.jsp">MOsDB</a><br />

<ul>
<li>MIPS Oryza sativa database</li>
</ul></li>
<li><a href="http://www.shigen.nig.ac.jp/rice/oryzabase/">Oryzabase</a><br />

<ul>
<li>Rice genetics and genomics</li>
</ul></li>
<li><a href="http://oryzatagline.cirad.fr/">Oryza Tag Line database</a><br />

<ul>
<li>T-DNA insertion mutants of rice</li>
</ul></li>
<li><a href="http://bis.zju.edu.cn/prin/">RRIN</a><br />

<ul>
<li>Oryza sativa protein-protein interactions network</li>
</ul></li>
</ul></li>

<li><h3 id="plant-promoter-and-regulatory-element-resources-植物启动子和调控元件资源">Plant Promoter and Regulatory Element Resources  植物启动子和调控元件资源</h3>

<ul>
<li><a href="http://arabidopsis.med.ohio-state.edu/">AGRIS</a><br />

<ul>
<li>Currently contains two databases, AtcisDB (Arabidopsis thaliana cis-regulatory database) and AtTFDB (Arabidopsis thaliana transcriptionfactor database).</li>
</ul></li>
<li><a href="http://www.athamap.de/">AthaMap</a><br />

<ul>
<li>A genome-wide map of putative transcription factor binding sites in Arabidopsis thaliana.</li>
</ul></li>
<li><a href="http://exon.cshl.org/cgi-bin/atprobe/atprobe.pl">AtProbe</a><br />

<ul>
<li>The Arabidopsis thaliana promoter binding element database, an aid to find binding elements and check data against the primary literature.</li>
</ul></li>
<li><a href="http://datf.cbi.pku.edu.cn/">DATF</a><br />

<ul>
<li>Database of Arabidopsis Transcription Factors (DATF) contains known and predicted Arabidopsis transcription factors with sequences and many other features including 3D structure templates, EST expression information, transcription factor binding sites and Nuclear Location Signals.</li>
</ul></li>
<li><a href="http://doop.abc.hu/">DoOP</a><br />

<ul>
<li>Databases of Orthologous Promoters, a database containing orthologous clusters of promoters from Homo sapiens, Arabidopsis thaliana and other organisms.</li>
</ul></li>
<li><a href="http://grassius.org/">GRASSIUS</a><br />

<ul>
<li>A public web resource composed by a collection of databases, computational and experimental resources that relate to the control of gene expression in the grasses, and their relationship with agronomic traits. GRASSIUS currently contains regulatory information on maize, rice, sorghum and sugarcane.</li>
</ul></li>
<li><a href="http://bioinformatics.psb.ugent.be/webtools/plantcare/html/">PlantCare</a><br />

<ul>
<li>Database of plant cis-acting regulatory elements and a portal to tools for in silico analysis of promoter sequences.</li>
</ul></li>
<li><a href="http://linux1.softberry.com/berry.phtml?topic=plantprom&amp;group=data&amp;subgroup=plantprom">PlantProm DB</a><br />
-Database with annotated, non-redundant collection of proximal promoter sequences for RNA polymerase II with experimentally determined transcription start sites (TSS) from various plant species.</li>
<li><a href="http://plntfdb.bio.uni-potsdam.de/v2.0/">PlantTFDB</a><br />

<ul>
<li>Plant Transcription Factor Database, an integrative plant transcription factor database that provides a web interface to access large (close to complete) sets of transcription factors of several plant species, currently encompassing Arabidopsis thaliana (thale cress), Populus trichocarpa (poplar), Oryza sativa(rice), Chlamydomonas reinhardtii and Ostreococcus tauri.</li>
</ul></li>
<li><a href="http://ppdb.agr.gifu-u.ac.jp/ppdb/cgi-bin/index.cgi">PPDB</a><br />

<ul>
<li>(Plant Promoter DB) Database that provides transcription start sites (TSS) and other structural information for Arabidopsis and rice promoters.</li>
</ul></li>
<li><a href="http://www.gene-regulation.com/pub/databases.html#transfac">Transfac</a><br />

<ul>
<li>Database on eukaryotic transcription factors, their genomic binding sites and DNA-binding profiles. Commercial site.
<br />
<br />
<br /></li>
</ul></li>
</ul></li>

<li><h3 id="databases-of-other-organisms-其他物种数据库">Databases of other Organisms  其他物种数据库</h3>

<ul>
<li><a href="http://www.genomesonline.org/cgi-bin/GOLD/index.cgi">GOLD</a><br />

<ul>
<li>Genomes OnLine Database, information on complete and ongoing genome projects</li>
</ul></li>
<li><a href="http://www.tigr.org/tigr-scripts/CMR2/CMRHomePage.spl">TIGR</a><br />

<ul>
<li>Comprehensive Microbial Resource</li>
</ul></li>
<li><a href="http://www.tigr.org/tdb/mdb/mdb.html">TIGR Microbial Database</a></li>
<li><a href="http://www.proteome.com/databases/">The Proteome Databases</a><br />

<ul>
<li>yeast, worm, &amp; human, good annotation</li>
</ul></li>
<li><a href="http://www.yeastgenome.org/">Saccharomyces Genome Database</a></li>
<li><a href="http://www.wormbase.org/">WormBase</a><br />

<ul>
<li>C. elegans</li>
</ul></li>
<li><a href="http://flybase.bio.indiana.edu/">FlyBase</a></li>
<li><a href="http://www.fruitfly.org/">Berkeley Drosophila Genome Project</a></li>
<li><a href="http://www.informatics.jax.org/">Mouse Genome Informatics</a></li>
<li><a href="http://www.arabidopsis.org/">The Arabidopsis Information Resource</a></li>
<li><a href="http://zfin.org/">ZFIN</a><br />

<ul>
<li>Zebrafish Information Network</li>
</ul></li>
<li><a href="http://dictybase.org/">DictyBase</a><br />

<ul>
<li>Dictyostelium discoideum</li>
</ul></li>
<li><a href="http://bmb.med.miami.edu/EcoGene/EcoWeb/">EcoGene</a><br />

<ul>
<li>E. coli</li>
</ul></li>
<li><a href="http://hiv-web.lanl.gov/">HIV sequence database</a>
<br />
<br />
<br /></li>
</ul></li>

<li><h3 id="genome-wide-analysis-基因组分析">Genome-wide Analysis  基因组分析</h3>

<ul>
<li><a href="http://mbgd.genome.ad.jp/">MBGD</a><br />

<ul>
<li>comparative analysis of completely sequenced microbial genomes</li>
</ul></li>
<li><a href="http://www.ncbi.nlm.nih.gov/COG/">COGs</a><br />

<ul>
<li>phylogenetic classification of orthologous proteins from complete genomes</li>
</ul></li>
<li><a href="http://www.bork.embl-heidelberg.de/STRING/">STRING</a><br />

<ul>
<li>detect whether a given query gene occurs repeatedly with certain other genes in potential operons</li>
</ul></li>
<li><a href="http://www.expasy.ch/tools/dna.html">Pedant</a><br />

<ul>
<li>automatic whole genome annotation</li>
</ul></li>
<li><a href="http://bioinfo.mbb.yale.edu/genome/">GeneCensus</a><br />

<ul>
<li>various whole genome comparisons
<br />
<br />
<br /></li>
</ul></li>
</ul></li>

<li><h3 id="protein-properties-蛋白质理化参数">Protein Properties  蛋白质理化参数</h3>

<ul>
<li><a href="https://web.expasy.org/compute_pi/">Compute pI/MW</a><br />

<ul>
<li>蛋白质等电点和分子量计算</li>
</ul></li>
<li><a href="https://web.expasy.org/protparam/">Protparam tool</a><br />

<ul>
<li>蛋白质综合理化参数计算</li>
</ul></li>
<li><a href="http://www.cbs.dtu.dk/services/SignalP/">SignalP</a><br />

<ul>
<li>蛋白质信号肽预测</li>
</ul></li>
<li><a href="https://embnet.vital-it.ch/software/TMPRED_form.html">TmPred</a><br />

<ul>
<li>蛋白跨膜结构预测</li>
</ul></li>
<li><a href="http://www.cbs.dtu.dk/services/NetNGlyc">NetNGlyc</a><br />

<ul>
<li>预测蛋白的糖基化位点
<br />
<br />
<br /></li>
</ul></li>
</ul>

<h3 id="protein-domains-蛋白质结构域">Protein Domains  蛋白质结构域</h3>

<ul>
<li><a href="http://www.compbio.dundee.ac.uk/jpred/">Jpred</a><br />

<ul>
<li>蛋白二级结构预测</li>
</ul></li>
<li><a href="https://npsa-prabi.ibcp.fr/cgi-bin/npsa_automat.pl?page=/NPSA/npsa_sopma.html">SOMPA</a><br />

<ul>
<li>蛋白质二级结构预测</li>
</ul></li>
<li><a href="http://genome.wustl.edu/Pfam/">Pfam</a><br />

<ul>
<li>alignments and hidden Markov models covering many common protein domains</li>
</ul></li>
<li><a href="http://smart.embl-heidelberg.de/">SMART</a><br />

<ul>
<li>analysis of domains in proteins</li>
</ul></li>
<li><a href="http://prodom.prabi.fr/prodom/current/html/home.php">ProDom</a><br />

<ul>
<li>protein domain database</li>
</ul></li>
<li><a href="http://www.ebi.ac.uk/interpro/">InterPro</a><br />

<ul>
<li>integration of Pfam, PRINTS, PROSITE, SWISS-PROT + TrEMBL</li>
</ul></li>
<li><a href="http://prosite.expasy.org/cgi-bin/prosite/PSScan.cgi">PROSITE</a><br />

<ul>
<li>database of protein families and domains</li>
</ul></li>
<li><a href="https://www.swissmodel.expasy.org/">SWISS-MODEL</a><br />

<ul>
<li>Protein 3D structure models</li>
</ul></li>
<li><a href="http://www.ncbi.nlm.nih.gov/Structure/">MMDB</a><br />

<ul>
<li>NCBI protein structure database</li>
</ul></li>
<li><a href="http://www.bioinf.man.ac.uk/dbbrowser/PRINTS/index.php">PRINTS Database</a><br />

<ul>
<li>groups of conserved motifs used to characterise protein families</li>
</ul></li>
<li><a href="http://blocks.fhcrc.org/">Blocks</a><br />

<ul>
<li>multiply aligned ungapped segments corresponding to the most highly conserved regions of proteins</li>
</ul></li>
<li><a href="http://www.tigr.org/TIGRFAMs/">TIGRFAMs</a><br />

<ul>
<li>yet more protein families based on Hidden Markov Models
<br />
<br />
<br /></li>
</ul></li>
</ul></li>

<li><h3 id="motif-and-pattern-search-in-sequences-序列模体和样式搜索">Motif and Pattern Search in Sequences  序列模体和样式搜索</h3>

<ul>
<li><a href="http://ccmbweb.ccv.brown.edu/gibbs/gibbs.html">Gibbs Motif Sampler</a><br />

<ul>
<li>identification of conserved motifs in DNA or protein sequences</li>
</ul></li>
<li><a href="http://arep.med.harvard.edu/mrnadata/mrnasoft.html">AlignACE Homepage</a><br />

<ul>
<li>gene regulatory motif finding</li>
</ul></li>
<li><a href="http://meme.nbcr.net/meme/intro.html">MEME</a><br />

<ul>
<li>motif discovery and search in protein and DNA sequences</li>
</ul></li>
<li><a href="http://www.cse.ucsc.edu/research/compbio/sam.html">SAM</a><br />

<ul>
<li>tools for creating and using Hidden Markov Models</li>
</ul></li>
<li><a href="http://www.ii.uib.no/~inge/Pratt.html">Pratt</a><br />

<ul>
<li>discover patterns in unaligned protein sequences</li>
</ul></li>
<li><a href="http://motif.gla.ac.uk/motif/index.html">Motivated Proteins</a><br />

<ul>
<li>a web facility for exploring small hydrogen-bonded motifs
<br />
<br />
<br /></li>
</ul></li>
</ul></li>

<li><h3 id="protein-3d-structure-蛋白质三维结构">Protein 3D Structure  蛋白质三维结构</h3>

<ul>
<li><a href="http://www.rcsb.org/pdb/">PDB</a><br />

<ul>
<li>protein 3D structure database</li>
</ul></li>
<li><a href="http://www.umass.edu/microbio/rasmol/">RasMol / Protein Explorer</a><br />

<ul>
<li>molecule 3D structure viewers</li>
</ul></li>
<li><a href="http://scop.mrc-lmb.cam.ac.uk/scop/">UCL BSM CATH classification</a></li>
<li><a href="http://www2.ebi.ac.uk/dali/fssp/fssp.html">FSSP</a><br />

<ul>
<li>fold classification based on structure-structure alignment of proteins</li>
</ul></li>
<li><a href="http://www.expasy.ch/swissmod/SWISS-MODEL.html">SWISS-MODEL</a><br />

<ul>
<li>homology modeling server</li>
</ul></li>
<li><a href="https://www.genesilico.pl/meta2/">Structure Prediction Meta-server</a></li>
<li><a href="http://zlab.bu.edu/k2">K2</a><br />

<ul>
<li>protein structure alignment</li>
</ul></li>
<li><a href="http://ekhidna.biocenter.helsinki.fi/dali_server/">DALI</a><br />

<ul>
<li>3D structure alignment server</li>
</ul></li>
<li><a href="http://swift.cmbi.ru.nl/gv/dssp/">DSSP</a><br />

<ul>
<li>defines secondary structure and solvent exposure from 3D coordinates</li>
</ul></li>
<li><a href="http://swift.cmbi.ru.nl/gv/hssp/">HSSP Database</a><br />

<ul>
<li>Homology-derived Secondary Structure of Proteins</li>
</ul></li>
<li><a href="https://www.predictprotein.org/">PredictProtein &amp; PHD</a><br />

<ul>
<li>predict secondary structure, solvent accessibility, transmembrane helices, and other stuff</li>
</ul></li>
<li><a href="http://www.compbio.dundee.ac.uk/www-jpred/">Jpred3</a><br />

<ul>
<li>protein secondary structure prediction</li>
</ul></li>

<li><p><a href="http://bioinf.cs.ucl.ac.uk/psipred/">PSIpred (&amp; MEMSAT &amp; GenTHREADER)</a></p>

<ul>
<li>protein secondary structure prediction (&amp; transmembrane helix prediction &amp; tertiary structure prediction by threading)
<br /></li>
</ul></li>

<li><p><strong>Structural analysis  结构分析</strong></p></li>
</ul>

<p><a href="http://pfam.sanger.ac.uk/">Pfam</a><br />
- A collection of protein families<br />
<a href="http://smart.embl-heidelberg.de/">SMART</a><br />
- Simple Modular Architecture Research Tool<br />
<a href="http://scop.mrc-lmb.cam.ac.uk/scop/">SCOP</a><br />
- Structural Classification of Proteins<br />
<a href="http://meme.sdsc.edu/meme4_1/intro.html">MEME</a><br />
- Motif-based sequence analysis tools<br />
<a href="http://www.compbio.dundee.ac.uk/www-jpred/index.html">Jpred3</a><br />
- A Secondary Structure Prediction Server<br />
<a href="http://bioinf.cs.ucl.ac.uk/psipred/">PSIPRED</a><br />
- A highly accurate method for protein secondary structure prediction<br />
<a href="http://clavius.bc.edu/~clotelab/DiANNA/">DiANNA</a><br />
- Cysteine state and Disulfide Bond partner prediction<br />
<a href="http://robetta.bakerlab.org/">Robetta</a><br />
- Full-chain protein structure prediction server</p>

<ul>
<li><strong>Molecular Graphics  分子图像</strong>
<br /></li>
</ul>

<p><a href="http://www.bioblender.eu/">BioBlender</a> <br />
- Open Source viewer that includes features for morphing proteins and visualization of lipophilic and electrostatic potentials.<br />
<a href="http://bragi2.helmholtz-hzi.de/">BRAGI</a> <br />
- A protein visualization and modeling program <a href="http://www.ccp4.ac.uk/MG/">CCP4mg</a> <br />
- Create beautiful publication quality images and movies. Users can superpose and analyse structures as well. The program runs &lsquo;out of the box&rsquo; on Linux, MacOSX and Windows platforms.<br />
<a href="http://www.cgl.ucsf.edu/chimera">Chimera</a> <br />
- Interactive molecular modeling system, free to academic/non-profit; displays multiple sequence alignments and associated structures, atom-type and H-bond identification, molecular dynamics trajectories (AMBER format), and offers ligand-screening interface (DOCK), filter by number/position of H-bonds, and extensibility to create custom modules - for Windows, Linux, Mac OS X, IRIX, and Tru64 Unix<br />
<a href="http://www.ncbi.nlm.nih.gov/Structure/CN3D/cn3d.shtml">Cn3D</a> <br />
- Simultaneously displays structure, sequence, and alignment, with annotation and alignment editing features, for use with 3-D structures from NCBI&rsquo;s Entrez; available for Windows, Macintosh, and Unix<br />
<a href="http://www.crystalmaker.com/crystalmaker/">CrystalMaker</a> <br />
- A program for building, displaying and manipulating all kinds of crystal and molecular structures.<br />
<a href="http://epmv.scripps.edu/">ePMV</a> <br />
Embedded Python Molecular Viewer (ePMV) is an open-source plug-in that runs molecular modeling software directly inside of professional 3D animation applications<br />
<a href="https://fold.it/portal/">Foldit</a> <br />
- Foldit is a crowdsourcing computer game based on protein modeling.<br />
<a href="http://www.pirx.com/iMol/">iMol</a> <br />
- Open GL graphics program displays small, large, and multiple molecules; measures distances and angles, superimposes structures, calculates RMSD between atom coordinates, structurally aligns chains, and displays dynamics trajectories. For Mac OS X incl. 10.2<br />
<a href="http://www.jmol.org/">Jmol</a> <br />
- Jmol is a free, open source molecule viewer for students, educators, and researchers in chemistry and biochemistry. It is cross-platform, running on Windows, Mac OS X, and Linux/Unix systems.<br />
<a href="http://kinemage.biochem.duke.edu/">Mage and Kinemages</a><br />
- Interactive molecular display for research and educational uses. Free, open source for Windows and Mac (OSX or PPC), Unix, and Linux. A Java version does 3-D Web display without plug-ins.<br />
<a href="http://www.chemaxon.com/">Marvin</a><br />
- Marvin is a collection of tools for drawing, displaying and characterizing chemical structures, queries, macromolecules and reactions for all operating systems, web pages and custom applications.<br />
<a href="http://cm2.cellmicrocosmos.org/">MembraneEditor</a><br />
- Interactively generate heterogeneous PDB-based membranes with varying lipid compositions and semi-automatic protein placement. Supports membrane patches and vesicles, microdomains as well as stacking of monolayer and/or bilayer membranes.<br />
<a href="http://molecule-world.com/dwb/products/molecule-world">Molecule World</a><br />
- an iPad application for viewing and manipulating 3D chemical and molecular structures. Structures can be downloaded and displayed from the PubChem, PDB, and NCBI structure databases together with the sequences for proteins and nucleic acids. Structures can be drawn as tubes, ball and stick, or space filling modes. Coloring options include residue, charge, hydrophobicity, rainbow, and molecule. Parts of structures can be hidden or displayed with mixed coloring and drawing modes.<br />
<a href="http://molecule-world.com/dwb/molecule-world-iphone-apple-watch">Molecule World for iPhone</a><br />
- Molecule World for iPhone can be used on the iPhone or iPod touch to display and manipulate 3D chemical and molecular structures from the PubChem, PDB, or NCBI structure databases. Drawing options include ball and stick, space fill, and ball and stick modes. Coloring options include rainbow, residue, charge, hydrophobicity, and molecule. Proteins, nucleic acids, and heterogens can be displayed in different modes.<br />
<a href="http://digitalworldbiology.com/dwb/DNA-binding-lab">Molecule World DNA Binding Lab<br />
</a>- A classroom ready iPad application for exploring the ways chemicals and proteins bind to DNA. The DNA Binding Lab uses Molecule World?s rendering engine and display features to highlight different molecules and understand how they intact. The DNA Binding Lab includes instructions, three examples, and 40 unknowns that can be assigned to students. Photo sharing capabilities allow students to share their work with teachers to aid with assessment.<br />
<a href="http://www.sunsetlakesoftware.com/molecules">Molecules</a><br />
- An iPhone application for PDB structures<br />
<a href="http://www.avatar.se/molscript/">MolScript<br />
</a>- A program for displaying structures in both detailed and schematic formats and writing images in various formats for Unix<br />
a style=&ldquo;color: #45962E&rdquo; href=&ldquo;<a href="http://www.molviz.org/&quot;">http://www.molviz.org/&quot;</a> target=&rdquo;_blank&rdquo;&gt;MolviZ.org<br />
- Free, interactive visualization tutorials<br />
<a href="http://www.zmmsoft.com/">MVM<br />
</a>- Molecular Visualization Program and GUI of ZMM. MVM is a free molecular viewer that can be used to display protein, nucleic acids, oligosacharides, small and macromolecules. It has an intuitive interface. In addition to being a molecular viewer, it is the user interface of a very powerful molecular mechanics engine (ZMM).<br />
<a href="http://mgltools.scripps.edu/">PMV (Python Molecular Viewer)</a> <br />
- An interactive molecular visualization and modeling environment for manipulation and viewing of multiple molecules.<br />
<a href="http://birg.cs.wright.edu/pocketmol/pocketmol.html">PocketMol</a> <br />
- Program to view and manipulate PDB files on a PocketPC <br />
POLYVIEW <br />
<a href="http://polyview.cchmc.org/">POLYVIEW-2D</a><br />
- Protein structure annotation using sequence profiles<br />
<a href="http://polyview.cchmc.org/polyview3d.html">POLYVIEW-3D</a><br />
- Versatile annotation and high quality visualization of macromolecular structures<br />
<a href="http://polyview.cchmc.org/conform.html">POLYVIEW-MM</a><br />
- Analysis and visualization of macromolecular motions<br />
<a href="http://www.proteinscope.com/">ProteinScope</a> <br />
- Free viewer to display and manipulate PDB files and create animations and slides of proteins for Windows. Online ordering of protein 3D prints in several color schemes.<br />
<a href="http://prosat.h-its.org/">Prosat</a> <br />
- Mapping protein sequence annotations onto a protein structure and visualizing them simultaneously with the structure.<br />
<a href="http://www.pymol.org/">PyMOL</a> <br />
- A free and open-source molecular graphics system for visualization, animation, editing, and publication-quality imagery. PyMOL is scriptable and can be extended using the Python language. Supports Windows, Mac OSX, Unix, and Linux<br />
<a href="http://qutemol.sourceforge.net/">QuteMol</a> <br />
- An open source (GPL), interactive, high quality molecular visualization system. QuteMol exploits the current GPU capabilites through OpenGL shaders to offers an array of innovative visual effects.<br />
<a href="http://www.bernstein-plus-sons.com/software/rasmol/">RasMol<br />
</a>- A free viewing system for PDB coordinate files that runs on Mac (PPC), Windows, Unix, and Linux systems. <a href="http://www.openrasmol.org/">Open source versions</a> are also available.<br />
<a href="http://skuld.bmsc.washington.edu/raster3d/">Raster3D<br />
</a>- A set of tools for generating high quality raster images of proteins or other molecules. Freeware for Mac OSX, Windows, Unix, and Linux<br />
<a href="http://www.geneinfinity.org/rastop">RasTop (v. 2.0)</a> <br />
- A free user-friendly graphical interface to RasMol molecular visualization software (v. 2.7.2.1), available for Windows and Linux<br />
<a href="http://www.msg.ucsf.edu/local/programs/ribbons/ribbons.html">Ribbons<br />
</a>- A program for molecular illustration and error analysis, for for Mac OSX, Windows, Unix, and Linux<br />
<a href="http://biojava.org/wiki/RCSB_Viewers:About">RCSB MBT Viewers</a> <br />
- The MBT toolkit is a framework that allows to create various viewers. It is used for 4 different viewers on the RCSB PDB web site.<br />
<a href="http://rmscopii.sourceforge.net/">RmscopII</a> <br />
- A Tcl/Tk script responsible to redirect PDB files or RasMol scripts to multiple RasMol sessions; can be used as a Web browser helper application or as a standalone program for Mac (OSX or PPC), Windows, or Unix<br />
<a href="http://www.schrodinger.com/">Schrödinger Product Suites</a> <br />
- Schrödinger&rsquo;s full product offerings range from general molecular modeling programs to a comprehensive suite of drug design software, as well as a state-of-the-art suite for materials research. All products are run with Maestro, a unified interface for all Schrödinger software, which is available for Mac, Windows, and Linux.<br />
<a href="http://www.spadeweb.org/">SPADE</a> <br />
- The Structural Proteomics Application Development Environment (SPADE) provides community tools for development and deployment of essential structure and sequence equipment. Includes a chemical probing suite to support experimental verification of predicted structural models. Written in Python with scripting tools available. Runs on Windows, Linux and Mac.<br />
<a href="http://3d-alignment.eu/">STRAP</a> <br />
- Align proteins by sequence and 3D structure.<br />
<a href="http://spdbv.vital-it.ch/">Swiss PDB viewer</a><br />
- A 3D graphics and molecular modeling program for the simultaneous analysis of multiple models and for model-building into electron density maps. The software is available for Mac (OSX or PPC), Windows, Linux, or SGI<br />
<a href="http://ugene.unipro.ru/">UGENE</a> <br />
- A free and open-source tool with PDB format visualization support written in fast memory efficient C++ code. Supports Windows, Mac OSX, Unix, and Linux.<br />
<a href="http://www.ks.uiuc.edu/Research/vmd/">VMD<br />
</a>- VMD (Visual Molecular Dynamics) runs on many platforms including MacOS X, and several versions of Unix and Windows. VMD provides visualization, analysis, and Tcl/Python scripting features, and has recently added sequence browsing and volumetric rendering features. VMD is distributed free of charge.<br />
<a href="http://www.yasara.org/">YASARA</a> - A complete molecular graphics and modeling program, including interactive molecular dynamics simulations, structure determination, analysis and prediction, docking, movies and eLearning for Windows, Linux and MacOSX.<br />
<a href="http://www.al-nasir.com/www/Jamie/Zeus/">Zeus</a> - A molecular visualization tool that supports PDB, MOL, MOL2/SYBYL and XYZ file formats. The rendering engine can output high quality molecular graphics. Zeus provides a sequence search that can highlight within the molecular structure. Ramachandran plots of internal dihedral angles can be generated and exported. PDB files can be automatically downloaded from the RSCB PDB.</p></li>

<li><h3 id="phylogeny-taxonomy-进化与分类">Phylogeny &amp; Taxonomy  进化与分类</h3>

<ul>
<li><a href="http://tolweb.org/tree/phylogeny.html">The Tree of Life</a></li>
<li><a href="http://www.sp2000.org/">Species 2000</a><br />

<ul>
<li>index of the world&rsquo;s known species</li>
</ul></li>
<li><a href="http://treebase.org/treebase-web/home.html">TreeBASE</a><br />

<ul>
<li>a database of phylogenetic knowledge</li>
</ul></li>
<li><a href="https://www.megasoftware.net/">MEGA</a><br />

<ul>
<li>Molecular Evolutionary Genetics Analysis</li>
</ul></li>
<li><a href="http://abacus.gene.ucl.ac.uk/software/paml.html">PAML</a><br />

<ul>
<li>Phylogenetic Analysis by Maximum Likelihood</li>
</ul></li>
<li><a href="http://www.atgc-montpellier.fr/phyml/usersguide.php">PHYML</a><br />

<ul>
<li>a phylogeny software based on the maximum-likelihood principle</li>
</ul></li>
<li><a href="http://evolution.genetics.washington.edu/phylip.html">PHYLIP</a><br />

<ul>
<li>package of programs for inferring phylogenies</li>
</ul></li>
<li><a href="http://taxonomy.zoology.gla.ac.uk/rod/treeview.html">TreeView</a><br />

<ul>
<li>user friendly tree displaying for Macs &amp; Windows
<br />
<br />
<br /></li>
</ul></li>
</ul></li>

<li><h3 id="gene-prediction-基因预测">Gene Prediction  基因预测</h3>

<ul>
<li><a href="http://hollywood.mit.edu/GENSCAN.html">Genscan</a><br />

<ul>
<li>eukaryotes - Identification of complete gene structures in genomic DNA</li>
</ul></li>
<li><a href="http://www.softberry.com/berry.phtml?topic=fgenesh&amp;group=programs&amp;subgroup=gfind">Fgenesh</a><br />

<ul>
<li>the fastest ab initio HMM-based eukaryotic gene prediction program</li>
</ul></li>
<li><a href="http://opal.biology.gatech.edu/GeneMark/">GeneMark</a></li>
<li><a href="http://ccb.jhu.edu/software/glimmer/index.shtml">GLIMMER</a><br />

<ul>
<li>prokaryotes</li>
</ul></li>
<li><a href="http://lowelab.ucsc.edu/tRNAscan-SE/">tRNAscan - SE 1.21</a><br />

<ul>
<li>search for tRNA genes in genomic sequence</li>
</ul></li>
<li><a href="http://www.sanger.ac.uk/Software/formats/GFF/GFF_Spec.shtml">GFF (General Feature Format) Specification</a><br />

<ul>
<li>a standard format for genomic sequence annotation
<br />
<br />
<br /></li>
</ul></li>
</ul></li>

<li><h3 id="gene-structure-analysis-基因结构分析">Gene Structure Analysis  基因结构分析</h3>

<ul>
<li><a href="https://www.ncbi.nlm.nih.gov/orffinder/">ORFfinder</a><br />

<ul>
<li>通用真核生物ORF预测</li>
</ul></li>
<li><a href="http://hollywood.mit.edu/genomescan.html%22">GENOMESCAN</a><br />

<ul>
<li>脊椎动物、拟南芥和玉米ORF预测</li>
</ul></li>
<li><a href="https://www.ebi.ac.uk/Tools/seqstats/emboss_cpgplot/">CpGplot</a></li>
<li><a href="http://linux1.softberry.com/berry.phtml?topic=polyah&amp;group=programs&amp;subgroup=promoter">PolyAH</a><br />

<ul>
<li>转录终止信号预测</li>
</ul></li>
<li><a href="http://www.cbs.dtu.dk/services/Promoter/#opennewwindow">PromotorScan</a><br />

<ul>
<li>启动子预测</li>
</ul></li>
<li><a href="https://www.dna.affrc.go.jp/PLACE/?action=newplace">PLACE</a><br />

<ul>
<li>植物顺式调控元件</li>
</ul></li>
<li><a href="https://services.healthtech.dtu.dk/service.php?NetGene2-2.42">NetGene2</a><br />

<ul>
<li>剪切位点预测</li>
</ul></li>
<li><a href="https://www.ncbi.nlm.nih.gov/sutils/splign/splign.cgi">Splign</a><br />

<ul>
<li>剪切位点预测
<br />
<br />
<br /></li>
</ul></li>
</ul></li>

<li><h3 id="gene-expression-databases-基因表达数据库">Gene Expression Databases  基因表达数据库</h3>

<ul>
<li><a href="http://zlab.bu.edu/HugeSearch/">HuGE</a><br />

<ul>
<li>database of human gene expression using arrays</li>
</ul></li>
<li><a href="http://arep.med.harvard.edu/ExpressDB/">ExpressDB</a><br />

<ul>
<li>yeast and E. coli RNA expression data</li>
</ul></li>
<li><a href="http://smd.princeton.edu/index.shtml">Stanford Microarray Database</a></li>

<li><p><a href="http://www.ncbi.nlm.nih.gov/geo/" title="GEO">Gene Expression Omnibus</a> (GEO)</p></li>

<li><p><strong>Microarray analysis  微阵列分析</strong></p>

<p><a href="http://www.ncbi.nlm.nih.gov/geo/">Gene Expression Omnibus</a> (NCBI)<br />
<a href="http://www.ebi.ac.uk/microarray-as/ae/">ArrayExpress</a></p>

<ul>
<li>is a public archive for transcriptomics data, which is aimed at storing MIAME- and MINSEQE- compliant data in accordance with MGED recommendations.<br />
<a href="http://smd.princeton.edu/index.shtml">Stanford Microarray Database</a><br />
<a href="http://www.mged.org/">MGED</a><br /></li>
<li>an international organization of biologists, computer scientists, and data analysts that aims to facilitate biological and biomedical discovery through data integration.<br />
<a href="http://www.bioconductor.org/">Bioconductor</a><br /></li>
<li>an open source and open development software project for the analysis and comprehension of genomic data.
<br />
<br />
<br /></li>
</ul></li>
</ul></li>

<li><h3 id="gene-regulation-基因调控">Gene Regulation  基因调控</h3>

<ul>
<li><a href="https://trafac.cchmc.org/trafac/index.jsp">TRAFAC</a><br />

<ul>
<li>For identifying conserved and shared cis regulatory elements between a pair of genes.</li>
</ul></li>
<li><a href="http://cismols.cchmc.org/peak-web/index.jsp" title="CisMols">CisMols</a><br />

<ul>
<li>For identifying conserved and shared cis regulatory elements between a set of co-expressed genes.</li>
</ul></li>
<li><a href="http://www.epd.isb-sib.ch/">EPD</a><br />

<ul>
<li>eukaryotic promoter database</li>
</ul></li>
<li><a href="http://dbtss.hgc.jp/">DBTSS</a><br />

<ul>
<li>DataBase of Transcriptional Start Sites (human)</li>
</ul></li>
<li><a href="http://rulai.cshl.edu/SCPD/">SCPD</a><br />

<ul>
<li>Saccharomyces cerevisiae promoter database</li>
</ul></li>
<li><a href="http://www-biology.ucsd.edu/labs/Kadonaga/DCPD.html">DCPD</a><br />

<ul>
<li>Drosophila Core Promoter Database</li>
</ul></li>
<li><a href="http://regulondb.ccg.unam.mx/">RegulonDB</a><br />

<ul>
<li>a database on transcriptional regulation in E. coli</li>
</ul></li>
<li><a href="http://arep.med.harvard.edu/dpinteract/">DPInteract</a><br />

<ul>
<li>protein binding sites on E. coli DNA</li>
</ul></li>
<li><a href="http://www.genomatix.de/online_help/help_gems/PromoterInspector_help.html">PromoterInspector</a><br />

<ul>
<li>prediction of promoter regions in mammalian genomic sequences</li>
</ul></li>
<li><a href="http://www.genomatix.de/online_help/help_matinspector/matinspector_help.html">MatInspector</a><br />

<ul>
<li>search for transcription factor binding sites</li>
</ul></li>
<li><a href="http://zlab.bu.edu/~mfrith/cister.shtml">Cister</a><br />

<ul>
<li>cis-element cluster finder</li>
</ul></li>
<li><a href="http://zlab.bu.edu/zlab/gene.shtml">Gene regulatory Tools</a>
<br />
<br />
<br /></li>
</ul></li>

<li><h3 id="small-rna-microrna-小分子rna-微rna">Small RNA/MicroRNA  小分子RNA/微RNA</h3>

<ul>
<li><a href="http://www.microrna.org/">microRNA Targets &amp; Expression Profiles</a></li>
<li><a href="http://bis.zju.edu.cn/pmirkb/">PmirKB</a><br />

<ul>
<li>Plant microRNA Knowledge Base</li>
</ul></li>

<li><p>MicroRNA Target Prediction<br />
<a href="http://www.microrna.org/microrna/home.do">miRanda</a><br />
— miRNA target prediction for human, drosophila and zebrafish genomes<br />
<a href="http://www.mirbase.org/">miRBase</a><br />
— a comprehensive repository for miRNAs and their predicted targets<br />
<a href="http://mirdb.org/miRDB/">miRDB</a><br />
— an online database for miRNA target prediction and functional annotations in animals<br />
<a href="http://mirnamap.mbc.nctu.edu.tw/">miRNAMap</a><br />
— a genomic maps of microRNA genes and their target genes in mammalian genomes<br />
<a href="http://www.mir2disease.org./">miR2Disease</a><br />
— a database providing comprehensive resource of miRNA deregulation in various human diseases<br />
<a href="http://diana.cslab.ece.ntua.gr/tarbase/">TarBase</a><br />
— a comprehensive database of experimentally supported animal microRNA targets<br />
<a href="http://pictar.mdc-berlin.de/">PicTar</a><br />
— microRNA targets for vertebrates, fly and nematodes<br />
<a href="http://www.targetscan.org/">TargetScan</a><br />
— a search for the presence of conserved sites that match the seed of each miRNA<br />
<a href="http://www.russell.embl.de/miRNAs/">Target Gene Prediction at EMBL</a><br />
— miRNA-Target predictions for Drosophila miRNAs</p></li>

<li><p><strong>microRNA Expression Databases  微RNA表达数据库</strong></p>

<p><a href="http://www.microrna.org/microrna/getExprForm.do">microRNA.org</a><br />
— predicted microRNA targets &amp; target downregulation scores. Experimentally observed expression patterns<br />
<a href="http://202.38.126.151/hmdd/mirna/md/">HMDD</a><br />
— Human MicroRNA Disease Database (HMDD) is a database that contains the experimentally supported miRNA-disease association data, which are manually curated from publications. The dysfunction evidence or miRNAs<br />
and literature PubMed ID are also given<br />
<a href="http://202.38.126.151/hmdd/mirna/tf/">TransmiR</a><br />
— a web query-driven database integrating the experimentally supported transcription factor and miRNA regulatory relations</p></li>

<li><p><strong>RNA Secondary Structure Prediction  RNA二级结构预测</strong></p>

<p><a href="http://diana.cslab.ece.ntua.gr/microT/">DIANA MicroTest</a><br />
— a prediction of miRNA-mRNA interaction<br />
<a href="http://mfold.bioinfo.rpi.edu/">mfold</a><br />
— tools for predicting the secondary structure of RNA and DNA, mainly by using thermodynamic methods<br />
<a href="http://bioinfo.uni-plovdiv.bg/microinspector/">microInspector</a><br />
—a web tool for detection of miRNA binding sites in an RNA sequence<br />
<a href="http://www.mirnainfo.com/EnergyCalc.aspx">miRNA Bioinfor</a><br />
—miRNA End Energy calculator which takes miRNA duplex to calculate free energy for 5 base pairs at one end plus a dangling nucleotide<br />
<a href="http://mirrim.ncrna.org/">miRRim</a><br />
— a method for detecting miRNA foldbacks based on hidden Markov model (HMM)<br />
<a href="http://mxscarna.ncrna.org/">MXSCARNA</a><br />
— a multiple alignment tool for RNA sequences using progressive alignment based on pairwise structural alignment algorithm of SCARNA. Good for large scale analyses.<br />
<a href="http://bibiserv.techfak.uni-bielefeld.de/rnahybrid/">RNAhybrid</a><br />
— a tool for finding the minimum free energy hybridisation of a long and a short RNA</p></li>

<li><p><strong>MicroRNA Homologous Prediction  微RNA同源预测</strong></p>

<p><a href="http://groups.csail.mit.edu/pag/mirnaminer/">miRNAminer</a><br />
— a web-based tool used for homologous miRNA gene search in several species<br />
<a href="http://people.csail.mit.edu/akiezun/miRviewer/">miRviewer</a><br />
—a global view of homologous miRNA genes in many species<br />
<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2676257/pdf/1471-2105-10-105.pdf">RISCbinder</a><br />
— prediction of guide strand of microRNAs<br />
<a href="http://tagc.univ-mrs.fr/mireval/">Mireval</a><br />
— Sequence evaluation of microRNA properties</p></li>

<li><p><strong>MicroRNA Deep Sequencing  微RNA深度测序</strong></p>

<p><a href="http://bioinfo5.ugr.es/miRanalyzer/miRanalyzer.php">miRanalyzer</a><br />
— A microRNA detection and analysis tool for next-generation sequencing experiments<br />
<a href="http://ibis.tau.ac.il/miRNAkey/">miRNAkey</a><br />
— A software pipeline for the analysis of microRNA Deep Sequencing data<br />
<a href="http://www.mdc-berlin.de/en/research/research_teams/systems_biology_of_gene_regulatory_elements/projects/miRDeep/index.html">miRDeep</a><br />
— Discovering known and novel miRNAs from deep sequencing data</p></li>
</ul></li>

<li><h3 id="metabolic-gene-regulatory-signal-transduction-network-databases-代谢-基因调控和信号转导网络数据库">Metabolic, Gene Regulatory &amp; Signal Transduction Network Databases  代谢、基因调控和信号转导网络数据库</h3>

<ul>
<li><a href="http://www.genome.ad.jp/kegg/">KEGG</a><br />

<ul>
<li>Kyoto Encyclopedia of Genes and Genomes</li>
</ul></li>
<li><a href="http://david.abcc.ncifcrf.gov/" title="DAVID">DAVID</a><br />

<ul>
<li><strong>D</strong>atabase for <strong>A</strong>nnotation, <strong>V</strong>isualization and <strong>I</strong>ntegrated <strong>D</strong>iscovery - A useful server to for annotating microarray and other genetic data.</li>
</ul></li>
<li><a href="http://bind.ca/">BIND</a><br />

<ul>
<li>Biomolecular Interaction Network Database</li>
</ul></li>
<li><a href="http://ecocyc.org/">EcoCyc</a></li>
<li><a href="http://www.pathguide.org/">PathGuide</a><br />

<ul>
<li>A very useful collection of resources dealing primarily with pathways</li>
</ul></li>
<li><a href="http://www.grt.kyushu-u.ac.jp/spad/">SPAD</a><br />

<ul>
<li>Signaling Pathway Database</li>
</ul></li>
<li><a href="http://www.gene-regulation.com/pub/databases.html">Transpath</a></li>

<li><p><a href="http://dip.doe-mbi.ucla.edu/">DIP</a></p>

<ul>
<li>Database of Interacting Proteins
<br /></li>
</ul></li>

<li><p><a href="http://www.cellularsignaling.org/">Alliance for Cellular Signalling</a></p></li>

<li><p><strong>Signaling pathways  信号通路</strong></p>

<p><a href="http://www.signaling-gateway.org/">AfCS-Nature Signaling Gateway</a><br />
<a href="http://www.stke.org/">AAAS | Science&rsquo;s STKE</a><br />
<a href="http://www.biocarta.com/search/index.asp">BioCarta</a></p>

<ul>
<li>Charting Pathways of Life<br />
<a href="http://nrresource.org/">The Nuclear Receptor Resource</a><br />
<a href="http://bibiserv.techfak.uni-bielefeld.de/stcdb/">STCDB</a><br /></li>
<li>Signal Transduction Classification Database</li>
</ul></li>
</ul></li>

<li><h3 id="systems-biology-系统生物学">Systems Biology  系统生物学</h3></li>

<li><p><a href="http://www.nature.com/focus/systemsbiologyuserguide/appendices/glossary.html">A glossary of systems biology terms</a></p></li>

<li><p><a href="http://www.nature.com/focus/systemsbiologyuserguide/appendices/institutes.html">A list of institutes specializing in systems biology or related research</a></p></li>

<li><p><a href="http://www.nature.com/focus/systemsbiologyuserguide/appendices/textbooks.html">A list of textbooks of interest to systems biologists</a></p></li>

<li><p><a href="http://www.systemsbiology.org/">SystemsBiology.org</a></p>

<ul>
<li>Institute for Systems Biology</li>
</ul></li>

<li><p><a href="http://www.systems-biology.org/">Systems Biology.org</a></p>

<ul>
<li>A portal site for systems biology</li>
</ul></li>

<li><p><a href="http://sbml.org/">SBML</a></p>

<ul>
<li>Systems Biology Markup Language</li>
</ul></li>

<li><p><a href="http://www.informatik.uni-hamburg.de/TGI/PetriNets/">Petri net world</a></p>

<ul>
<li>Online Services for the International Petri Nets Community
<br /></li>
</ul></li>

<li><p><strong><a href="http://www.nigms.nih.gov/Initiatives/SysBio/Centers/">Systems biology-funded centers</a>  系统生物学中心</strong></p>

<p><a href="http://www.genomedynamics.org/">Center for Genome Dynamics at the Jackson Laboratory</a><br />
<a href="http://www.sysbio.harvard.edu/csb/about/index.html">Harvard University</a><br />
<a href="http://www.centerforsystemsbiology.org/">Institute for Systems Biology</a><br />
<a href="http://www.cdpcenter.org/">Massachusetts Institute of Technology</a><br />
<a href="http://quantbio.princeton.edu/">Princeton University</a><br />
<a href="http://raven.zoology.washington.edu/celldynamics/">University of Washington Friday Harbor Laboratories</a></p></li>

<li><p><strong>Gene List Annotation Tools  基因列表注释工具</strong></p>

<p><a href="http://www.broad.mit.edu/gsea/msigdb/index.jsp">MSigDB</a></p>

<ul>
<li>Molecular Signatures Database<br />
<a href="http://toppgene.cchmc.org/">ToppGene Suite</a><br /></li>
<li>Gene list functional enrichment and candidate gene prioritization<br />
<a href="http://www.pantherdb.org/">Panther</a><br /></li>
<li><strong>P</strong>rotein <strong>AN</strong>alysis <strong>TH</strong>rough <strong>E</strong>volutionary <strong>R</strong>elationships<br />
<a href="http://depts.washington.edu/l2l/">L2L</a><br />
<a href="http://fatigo.bioinfo.cipf.es/">Babelomics (FatiGO+)</a><br />
<a href="http://vortex.cs.wayne.edu/projects.htm">OntoExpress</a><br />
<br /></li>
</ul></li>

<li><h3 id="synthetic-biology-合成生物学">Synthetic Biology  合成生物学</h3>

<ul>
<li>#### DNA Tools  DNA工具
<br /></li>
</ul>

<p><a href="http://alpha.dmi.unict.it/~ctnyu/bbocus.html" title="http://alpha.dmi.unict.it/~ctnyu/bbocus.html">BBOCUS</a> (BackTranslation Based On Codon Usage Strategy) by Ferro and Purrello lab, a re-implementation of the algorithm in Graziano Pesole&rsquo;s BACKTR. It&rsquo;s based on cluster analysis (Complete Linkage algorithm), that requires a similarity matrix D containing distance between each pair of sequences of mRNA.<br />
<a href="https://benchling.com/" title="https://benchling.com">Benchling</a><br />
- by Benchling, Inc. Free online tools for vector editing, restriction analysis, primer search, multi-sequence alignment, and more.<br />
<a href="http://paris.chem.yale.edu/extinct.html" title="http://paris.chem.yale.edu/extinct.html">Biopolymer calculator</a><br />
- by Schepartz lab. Calculate extinction coefficients, Tm&rsquo;s, and base composition for your DNA or RNA; calculate amino acid composition and extinction coefficient for your protein.<br />
<a href="http://synbio.mit.edu/tools/clipboard.cgi" title="http://synbio.mit.edu/tools/clipboard.cgi">Clipboard</a> by Austin Che. Web tool for getting complement, reverse complement, translation and restriction enzyme analysis of a DNA sequence.<br />
<a href="https://moleculamaxima.com/" title="https://moleculamaxima.com">Cytostudio</a><br />
- by Molecula Maxima. An integrated development environment and a compiler for a high-level bio-programming language for Synthetic Biology. Based on iGEM conventions.<br />
<a href="http://helixweb.nih.gov/dnaworks/" title="http://helixweb.nih.gov/dnaworks/">DNAWorks</a><br />
- by Hoover and Lubkowski. A web tool for optimizing melting temperature during gene synthesis.<br />
<a href="http://synbio.mit.edu/tools/converter.cgi" title="http://synbio.mit.edu/tools/converter.cgi">File format converter</a><br />
- by Austin Che. Web tool for converting between sequence file formats.<br />
<a href="https://www.geneious.com/" title="https://www.geneious.com">Geneious</a> by Biomatters. Comprehensive suite of tools for molecular biology.<br />
<a href="http://www.genomecompiler.com/?utm_source=syntheticbiology.org/tools" title="http://www.genomecompiler.com/?utm_source=syntheticbiology.org/tools">Genome Compiler</a><br />
- is the industry&rsquo;s most user friendly genetic engineering design tool. It allows you to manipulate genetic information; from genes to plasmids to whole genomes. You can rapidly access extensive libraries of genetic parts, and easily order your final design from a variety of providers.<br />
<a href="http://www.genedesign.org/" title="http://www.genedesign.org/">GeneDesign</a><br />
- by Boeke lab. Collection of online (and some command line) tools for codon optimization and shuffling, restriction site editing, and so on.<br />
<a href="https://www.dna20.com/tools/genedesigner.php" title="https://www.dna20.com/tools/genedesigner.php">GeneDesigner</a><br />
- by DNA2.0. Combine genetic building blocks by drag-and-drop, codon optimize, restriction site editing, sequence oligo design etc.<br />
<a href="http://www.genocad.com/" title="http://www.genocad.com/">GenoCAD</a><br />
- is a design tool that uses collections or libraries of genetic parts and explicit design rules describing how these parts should be combined to engineer genetic constructs.<br />
<a href="http://tools.neb.com/NEBcutter2/index.php" title="http://tools.neb.com/NEBcutter2/index.php">NEB Cutter</a><br />
- by New England Biolabs, Inc.Tool for finding restriction sites, et cetera.<br />
<a href="http://www.evolvingcode.net/codon/sgd/index.php" title="http://www.evolvingcode.net/codon/sgd/index.php">Synthetic Gene Designer</a><br />
- by Gang Wu. A web platform that allows codon optimization to various extent. Compatible with non-standard genetic codes.<br />
<a href="http://www.invitrogen.com/content.cfm?pageid=10373" title="http://www.invitrogen.com/content.cfm?pageid=10373">Vector NTI</a><br />
- by Informax, Inc. Free-to-academics tool for sequence analysis and data management.<br />
<a href="http://j5.jbei.org/" title="http://j5.jbei.org">j5, DeviceEditor, and VectorEditor</a>online tools<br />
- j5: DNA assembly design automation for (combinatorial) flanking homology (e.g., SLIC/Gibson/CPEC/SLiCE/yeast) and type IIs-mediated (e.g., Golden Gate/FX cloning) assembly methods.<br />
- DeviceEditor: a visual DNA design canvas that serves as front-end for j5.<br />
- VectorEditor: a visual DNA editing and annotation tool.<br />
<a href="https://mcds.codeplex.com/" title="https://mcds.codeplex.com">MCDS</a><br />
- (Molecular Cloning Designer Simulator) by Zhenyu Shi, aims to set up a standard way for designing, describing and simulating molecular cloning and genetic engineering.<br />
-   #### RNA Tools  RNA工具</p>

<p><a href="http://www.ambion.com/techlib/append/na_mw_tables.html" title="http://www.ambion.com/techlib/append/na_mw_tables.html">Appendix</a><br />
- by Ambion, Inc. Website with many useful nucleic acid parameters.<br />
<a href="http://www.bioinfo.rpi.edu/applications/mfold/" title="http://www.bioinfo.rpi.edu/applications/mfold/">mFold</a><br />
- by Michael Zuker. is for predicting RNA and DNA folds, calculating Tm&rsquo;s and free energies.<br />
-   #### Protein Tools  蛋白质工具</p>

<p><a href="http://ncbi.nih.gov/Structure/CN3D/cn3d.shtml" title="http://ncbi.nih.gov/Structure/CN3D/cn3d.shtml">Cn3D</a><br />
- by NCBI. A helper application for your web browser that allows you to view 3-dimensional structures from NCBI&rsquo;s Entrez retrieval service. It doesn&rsquo;t read PDB files but can be more straightforward to use than DeepView.<br />
<a href="http://www.expasy.org/spdbv/" title="http://www.expasy.org/spdbv/">DeepView</a> by GlaxoSmithKline &amp; Swiss Institute of Bioinformatics. Awesome program for viewing and studying protein structure.<br />
<a href="http://us.expasy.org/" title="http://us.expasy.org/">ExPASy Proteomics server</a><br />
- by the Swiss Institute of Bioinformatics. Collection of links to many pages to calculate parameters of your favorite proteins.<br />
<a href="http://salilab.org/modeller/" title="http://salilab.org/modeller/">Modeller</a><br />
- by Sali Lab. For homology or comparative modeling of protein three-dimensional structures.<br />
<a href="http://www.scripps.edu/mb/barbas/zfdesign/zfdesignhome.php" title="http://www.scripps.edu/mb/barbas/zfdesign/zfdesignhome.php">Zinc Finger Tools</a> by Barbas Lab. Design Zinc Finger DNA binding proteins.<br />
-   #### CAD Tools  CAD工具</p>

<p><a href="http://www.tinkercell.com/" title="http://www.tinkercell.com">TinkerCell</a><br />
- by Deepak Chandran. Construct computational models using biological parts, cells, and modules.<br />
<a href="http://osslab.lifesci.warwick.ac.uk/Tinker.aspx" title="http://osslab.lifesci.warwick.ac.uk/Tinker.aspx">Metabolic Tinker</a><br />
- by Kent McClymont and Orkun Soyer. Construct thermodynamically feasible metabolic paths among user-defined compounds.<br />
-   #### General Tools  一般工具</p>

<p><a href="http://genolist.pasteur.fr/Colibri/" title="http://genolist.pasteur.fr/Colibri/">Colibri</a><br />
- by Institut Pasteur. <em>E. coli</em> genome site; get sequences, see the position of your gene in the chromosome, see the function of your gene, and other fun stuff. You can also search for protein sequences/motifs within the <em>E. coli</em> genome.<br />
<a href="http://parts.mit.edu/" title="http://parts.mit.edu/">Registry of Standard Biological Parts</a><br />
- by MIT. Open repository of BioBricks; the place for all your standard biological parts.<br />
<a href="https://public-registry.jbei.org/" title="https://public-registry.jbei.org">JBEI Registry</a><br />
- A site where you can explore the various features of the JBEI Registry software, and even get some work done! A DNA part, plasmid, microbial strain, and Arabidopsis Seed online repository with physical sample tracking capabilities.<br />
<a href="http://parpar.jbei.org/" title="http://parpar.jbei.org">PaR-PaR Laboratory Automation Platform</a> allows researchers to use liquid-handling robots effectively, enabling experiments that would not have been considered previously. After minimal training, a biologist can independently write complicated protocols for a robot within an hour.</p></li>

<li><h3 id="other-databases-annotations-ontologies-consortia-etc">Other Databases (Annotations, Ontologies, Consortia, etc.)</h3>

<p>其他数据库（注释、本体、协会）</p>

<ul>
<li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=gene" title="Entrez Gene">Entrez Gene</a><br />

<ul>
<li>Gene provides a unified query environment for genes defined by sequence and/or in NCBI&rsquo;s Map Viewer. You can query on names, symbols, accessions, publications, GO terms, chromosome numbers, E.C. numbers, and many other attributes associated with genes and the products they encode.Replaces LocusLink.</li>
</ul></li>
<li><a href="http://www.genenames.org/">HUGO&rsquo;s Human Gene Nomenclature</a></li>
<li><a href="http://www.geneontology.org/">Gene Ontology Consortium</a> -  a controlled vocabulary of eukaryotic gene roles</li>
<li><a href="http://obofoundry.org/" title="OBO">Open Biological Ontologies</a><br />

<ul>
<li>an umbrella web address for well-structured controlled vocabularies for shared use across different biological domains.</li>
</ul></li>
<li><a href="http://biom3.univ-lyon1.fr/acuts/ACUTS_home.html">ACUTS</a><br />

<ul>
<li>compilation of Ancient Conserved UnTranslated Sequences</li>
</ul></li>
<li><a href="http://utrdb.ba.itb.cnr.it/">UTR database</a></li>
<li><a href="http://www.expasy.ch/enzyme/">ENZYME</a><br />

<ul>
<li>enzyme nomenclature database</li>
</ul></li>
<li><a href="http://www.brenda.uni-koeln.de/">BRENDA</a><br />

<ul>
<li>enzyme database</li>
</ul></li>
<li><a href="http://www.tcdb.org/">TC-DB</a><br />

<ul>
<li>comprehensive classification of membrane transport proteins</li>
</ul></li>
<li><a href="http://snp.cshl.org/">The SNP Consortium</a></li>
<li><a href="http://www.methdb.de/">MethDB</a><br />

<ul>
<li>DNA methylation database</li>
</ul></li>
<li><a href="http://linux1.softberry.com/spldb/SpliceDB.html">SpliceDB</a><br />

<ul>
<li>canonical and non-canonical splice site sequences in mammalian genes</li>
</ul></li>
<li><a href="http://www.neb.com/neb/inteins.html">InBase</a><br />

<ul>
<li>intein database</li>
</ul></li>
<li><a href="http://www.imageconsortium.org/">The I.M.A.G.E. Consortium</a></li>
<li><a href="http://rebase.neb.com/rebase/rebase.html">REBASE</a><br />

<ul>
<li>restriction enzyme database</li>
</ul></li>
<li><a href="http://www.gnf.org/" title="Novartis">Genomics Institute of the Novartis Research Foundation</a></li>
<li><a href="http://www.bio-computing.org/">Bio-computing.org</a><br />
Bioinformatics, Databases and Software for Medicine.</li>
<li><a href="https://www.intomics.com/">Intomics</a><br />

<ul>
<li>from data to biology
<br />
<br />
<br /></li>
</ul></li>
</ul></li>

<li><h3 id="miscellaneous-tools-其他工具">Miscellaneous Tools  其他工具</h3>

<ul>
<li><a href="https://wiki.nbic.nl/index.php/NGS_Tools">NGS Tools</a> (nbic)</li>
<li><a href="http://www.ncbi.nlm.nih.gov/projects/gbench/" title="NCBI Gwb">NCBI Genome Workbench</a><br />

<ul>
<li>NCBI Genome Workbench is an integrated application for viewing and analyzing sequence data. With Genome Workbench, you can view data in publically available sequence databases at NCBI, and mix this data with your own private data.</li>
</ul></li>
<li><a href="http://www.sanger.ac.uk/resources/">Sanger Resources</a><br />

<ul>
<li>tools and databases</li>
</ul></li>
<li><a href="http://www.repeatmasker.org/">Repeatmasker</a><br />

<ul>
<li>mask repetitive elements in DNA sequences</li>
</ul></li>
<li><a href="http://tandem.bu.edu/trf/trf.html">Tandem Repeats Finder</a></li>
<li><a href="http://www.tbi.univie.ac.at/~ivo/RNA/">Vienna RNA Package</a><br />

<ul>
<li>RNA secondary structure prediction</li>
</ul></li>
<li><a href="http://mfold.rna.albany.edu/">mfold</a><br />

<ul>
<li>RNA secondary structure prediction</li>
</ul></li>
<li><a href="http://www.ebi.ac.uk/Tools/seqstats/emboss_cpgplot/">CpG Islands</a><br />

<ul>
<li>EMBOSS Cpgplot-Identify and plot CpG islands in nucleotide sequence(s)</li>
</ul></li>
<li><a href="http://atgpr.dbcls.jp/">ATGpr</a><br />

<ul>
<li>prediction of translation start sites in cDNA sequences</li>
</ul></li>
<li><a href="http://psort.nibb.ac.jp/">PSORT</a><br />

<ul>
<li>prediction of protein sorting signals and transmembrane helices</li>
</ul></li>
<li><a href="http://www.cbs.dtu.dk/services/">CBS Prediction Servers</a><br />

<ul>
<li>prediction of protein subcellular localization and various sites in protein and nucleotide sequences</li>
</ul></li>
<li><a href="http://www.expasy.ch/tools/pi_tool.html">Compute pI/Mw Tool</a></li>
<li><a href="http://www.expasy.ch/tools/dna.html">Translate Tool</a></li>
<li><a href="http://bioweb.pasteur.fr/seqanal/interfaces/melting.html">Melting</a><br />

<ul>
<li>calculate melting temperature for nucleic acid duplexes</li>
</ul></li>
<li><a href="http://hydra.icgeb.trieste.it/dna/bend_it.html">bend.it</a><br />

<ul>
<li>calculate curvature and bendability of a DNA sequence</li>
</ul></li>
<li><a href="http://www.firstmarket.com/cutter/cut2.html">webcutter</a><br />

<ul>
<li>detect restriction enzyme cutting sites in DNA sequences</li>
</ul></li>
<li><a href="http://bioinfo.ut.ee/primer3-0.4.0/">Primer3</a><br />

<ul>
<li>pick primers from a DNA sequence</li>
</ul></li>
<li><a href="http://cse.niaes.affrc.go.jp/miwa/probcalc/index.html">Probability Distribution Calculators</a><br />

<ul>
<li>normal, chi square, t, F, etc.
<br />
<br />
<br /></li>
</ul></li>
</ul></li>

<li><h3 id="computational-resources-计算资源">Computational Resources  计算资源</h3>

<ul>
<li><a href="http://sourceforge.net/" title="SourceForge">SourceForge</a><br />

<ul>
<li>SourceForge.net is the world&rsquo;s largest Open Source software development website, with the largest repository of Open Source code and applications available on the Internet. SourceForge.net provides free services to Open Source developers.</li>
</ul></li>
<li><a href="http://www.w3.org/">W3C</a><br />

<ul>
<li>World Wide Web Consortium, definitive reference for HTML and other WWW stuff</li>
</ul></li>
<li><a href="http://httpd.apache.org/docs-project/">Apache web server documentation</a></li>
<li><a href="http://www.runoob.com/python/python-tutorial.html">Python中文基础教程</a></li>
<li><a href="http://www.php.net/">PHP information</a></li>
<li><a href="http://www.stars.com/">Web Developer&rsquo;s Virtual Library</a><br />

<ul>
<li>encyclopedia of web design tutorials, articles and discussions</li>
</ul></li>
<li><a href="http://www.hwg.org/">HTML Writers Guild</a></li>
<li><a href="http://www.cpan.org/">CPAN</a><br />

<ul>
<li>PERL modules</li>
</ul></li>
<li><a href="http://www.bioperl.org/">bioperl</a><br />

<ul>
<li>bioinformatics related PERL modules</li>
</ul></li>
<li><a href="http://www.sgi.com/tech/stl/">C++ Standard Template Library Programmer&rsquo;s Guide</a></li>
<li><a href="https://www.gnu.org/software/libc/">GNU C Library Reference</a></li>
<li><a href="http://www.eskimo.com/~scs/cclass/cclass.html">C Tutorial</a></li>
<li><a href="http://java.sun.com/docs/books/tutorial/">Java Tutorial</a></li>
<li><a href="http://xlinux.nist.gov/dads/">Dictionary of Algorithms, Data Structures, and Problems</a></li>
<li><a href="http://dsl.org/cookbook/cookbook_toc.html">The Linux Cookbook</a></li>
<li><a href="http://d3js.org/">Data-Driven Documents (D3.js)</a></li>
</ul></li>

<li><h3 id="human-diseases-and-cancer-databases-人类疾病与癌症数据库">Human Diseases and Cancer Databases  人类疾病与癌症数据库</h3>

<ul>
<li><a href="https://www.ncbi.nlm.nih.gov/omim">OMIM</a><br />

<ul>
<li>a comprehensive, authoritative compendium of human genes and genetic phenotypes that is freely available and updated daily.</li>
</ul></li>
<li><a href="http://www.w3.org/">MalaCards</a><br />

<ul>
<li>an integrated database of human maladies and their annotations,modeled on the architecture and richness of the popular GeneCards databse of human gene.</li>
</ul></li>
<li><a href="https://cancergenome.nih.gov/">TCGA</a><br />

<ul>
<li>(The Cancer Genome Atlas) a collaboration between the National Cancer Institute(NIC) And the National Human Genome Research Institute (NHGRI)that has generated comprehensive,mutil-dimensional maps of the key genomic changes in 33 types of cancer.</li>
</ul></li>
<li><a href="https://portals.broadinstitute.org/ccle">CCLE</a><br />

<ul>
<li>(Cancer Cell Line Encycclopedia) a collaboration between the Board Institute,and the Novartis Institute for Biomedical Research and its Genomics Institute of the Novartis Research Foundation to conduct a detailed genetic and pharmacologic characterization of a large panel of human cancer models, to develop integrated computational analyses that link distinct pharmacologic vulnerabilities to genomic patters and to translate cell line integrative genomics into cancer patient stratiffication.The CCLE provides public access to genomic data,analysis and visualization for about 1000 cell lines.</li>
</ul></li>
<li><a href="http://cgap.nci.nih.gov/">Cancer Genome Anatomy Project</a></li>
<li><a href="http://cancergenome.nih.gov/">The Cancer Genome Atlas</a></li>
<li><a href="http://www.sciencemag.org/content/337/6092/282.summary">Cancer Genetics With an Edge</a></li>
<li><a href="http://www.youtube.com/watch?v=KE8TY0gT--g">Cancers Genomes and their Implications for Curing Cancer</a> by <a href="http://www.hopkinsmedicine.org/kimmel_cancer_center/experts/Laboratory_Scientists/detail/6424146D144F331F200D784A751851DB/Bert_Vogelstein">Bert Vogelstein</a></li>
<li><a href="https://epi.grants.cancer.gov/cancer_risk_prediction/">Cancer Risk Prediction Models and Assessment</a>
<br />
<br />
<br /></li>
</ul></li>

<li><h3 id="medical-image-repositorie-医学影像库">Medical Image Repositorie  医学影像库</h3>

<ul>
<li><a href="https://nihcc.app.box.com/v/ChestXray-NIHCC">NIH</a><br />

<ul>
<li>Database of 100,000 Chest X-Ray images, associated data, and diagnoses</li>
</ul></li>
<li><a href="https://www.dicomstandard.org/">DICOM</a><br />

<ul>
<li>(Digital Imaging and Communications in Medicine)the international standard to transmit, store, retrieve, print, process, and display medical imaging information.</li>
</ul></li>
<li><a href="http://www.cancerimagingarchive.net/">The Cancer Imaging Archive (TCIA)</a></li>
<li><a href="http://www.oasis-brains.org/">OASIS</a><br />

<ul>
<li>Cross-sectional MRI Data in Young, Middle Aged, Nondemented and Demented Older Adults; Longitudinal MRI Data in Nondemented and Demented Older Adults</li>
</ul></li>
<li><a href="http://adni.loni.usc.edu/">ADNI</a><br />

<ul>
<li>Alzheimer’s Disease Neuroimaging Initiative (ADNI) unites researchers with study data as they work to define the progression of Alzheimer’s disease. ADNI researchers collect, validate and utilize data such as MRI and PET images, genetics, cognitive tests, CSF and blood biomarkers as predictors for the disease.</li>
</ul></li>
<li><a href="https://fitbir.nih.gov/">FITBIR</a><br />

<ul>
<li>The Federal Interagency Traumatic Brain Injury Research (FITBIR) informatics system: MRI, PET, Contrast, and other data on a range of TBI conditions</li>
</ul></li>
<li><a href="http://cecas.clemson.edu/~ahoover/stare/">STARE</a>:<br />

<ul>
<li>STructured Analysis of the Retina: This research concerns a system to automatically diagnose diseases of the human eye.</li>
</ul></li>
<li><a href="http://cancer.digitalslidearchive.net/">Cancer Digital Slide Archive</a><br />

<ul>
<li>Whole-slide images from The Cancer Genome Atlas&rsquo;s (TCGA) glioblastoma multiforme (GBM) samples</li>
</ul></li>
<li><a href="http://www.cancerimagingarchive.net/">The Cancer Imaging Archive</a><br />

<ul>
<li>The image data in The Cancer Imaging Archive (TCIA) is organized into purpose-built collections of subjects. The subjects typically have a cancer type and/or anatomical site (lung, brain, etc.) in common.</li>
</ul></li>
<li><a href="http://lbam.med.jhmi.edu/">Johns Hopkins Medical Institute</a><br />

<ul>
<li>DTI Atlases: adults, children, &hellip;</li>
</ul></li>

<li><p><a href="http://www.civm.duhs.duke.edu/SharedData/DataSupplements.htm">Duke Center for In Vivo Microscopy</a></p>

<ul>
<li>Small animal MRI, CT, &hellip;
<br /></li>
</ul></li>

<li><p><a href="http://www.insight-journal.org/midas/"><strong>MIDAS</strong></a></p></li>
</ul>

<p><a href="http://www.insight-journal.org/midas/community/view/17">National Alliance for Medical Image Computing (NAMIC)</a>: Lupus white matter lesions, Brain MRI: 2-4 years old, Prostate<br />
<a href="http://www.insight-journal.org/midas/community/view/15">NLM: Imaging Methods Assessment and Reporting</a>: Liver tumors with segmentations<br />
<a href="http://www.insight-journal.org/midas/community/view/21">100 Healthy Brain MRI: 18-90 years old</a><br />
-   <a href="http://archive.ics.uci.edu/ml/">UCI Machine Learning Repository</a><br />
    - The father of internet data archives for all forms of machine learning.
-   <a href="http://homepages.inf.ed.ac.uk/rbf/CVonline/Imagedbase.htm#biomed">Computer Vision Online Image Archive</a><br />
    - Large listing of multiple databases in computer vision and biomedical imaging
-   <a href="http://www.via.cornell.edu/databases/">Cornell Visualization and Image Analysis (VIA) group</a><br />
    - Provides a list of available databases, many of which are also listed here.
-   <a href="http://www.library.uthscsa.edu/find/databases.cfm?Category=Image%20Collections">UT Health Science Center Image Collections:</a><br />
    - List of medical images, atlases, and databases available on the web.
-   <a href="http://www.omnimedicalsearch.com/image_databases.html">OmniMedicalSearch.com: Medical Image Databases &amp; Libraries</a>
-   <a href="http://marathon.csee.usf.edu/Mammography/Database.html">Digital Database for Screening Mammography (DDSM)</a><br />
    -  Large collection with normal and abnormal findings and ground truth.
-   <a href="http://www.isi.uu.nl/Research/Databases/DRIVE/">Digital Retinal Images for Vessel Extraction (DRIVE)</a><br />
    - Digital images and expert segmentations of retinal vessels.
-   <a href="http://www.jsrt.or.jp/web_data/english03.php">Japanese Society of Radiological Technology (JSRT) Database</a><br />
    - Digital Chest X-ray images with lung nodule locations, ground truth, and controls.
-   <a href="http://www.isi.uu.nl/Research/Databases/SCR/">Segmentation in Chest Radiographs (SCR) database</a><br />
    - Digital Chest X-ray images with segmentations of lung fields, heart, and clavicles.
-   <a href="http://www.via.cornell.edu/crpf.html">Public Lung Database to Address Drug Response</a><br />
    - Well documented chest CT images.
-   <a href="http://peipa.essex.ac.uk/info/mias.html">Mammographic Image Analysis Society (mini-MIAS) Database</a><br />
    - Mammographic images and markup.
-   <a href="http://www2.it.lut.fi/project/imageret/diaretdb1/">Standard Diabetic Retinopathy Database (DIARETDB1)</a><br />
    - Digital retinal images for detecting and quantifying diabetic retinopathy.
-   <a href="http://spineweb.digitalimaginggroup.ca/">SpineWeb</a><br />
    - SpineWeb is an online collaborative platform for everyone interested in research on spinal imaging and image analysis.
-   <a href="https://oai.epi-ucsf.org/datarelease/">OAinitiative</a><br />
    - MR data of Hips, knees and other sites affected by OA
-   <a href="https://pediatricmri.nih.gov/nihpd/info/index.html">MR pediatric repository</a><br />
    - MR brain
-   <a href="http://facebase.org/">Facebase</a><br />
    - 3D photography 
-   <a href="http://www.synapse.org/">Synapse</a><br />
    - NIH-funded datasets.<br />
    For example, <a href="https://www.synapse.org/#!Synapse:syn4152456">from UNC</a></p></li>

<li><h3 id="bioinformatics-on-line-course-materials-and-tutorials-生物信息学在线课程教材">Bioinformatics on-line course materials and tutorials  生物信息学在线课程教材</h3>

<ul>
<li><p><a href="http://lectures.molgen.mpg.de/online_lectures.html"><strong>Online Lectures on Bioinformatics</strong></a></p></li>

<li><p><a href="http://www.med.nyu.edu/rcr/rcr/course/index.html"><strong>Using Computers in Molecular Biology</strong></a></p></li>

<li><p><strong>Intro to bioinformatics and computational biology</strong></p></li>
</ul>

<p><a href="http://www.cs.technion.ac.il/~cs236606/">Introduction to Bioinformatics</a> (Technion - Israel Institute of Technology)<br />
<a href="http://www.biochem.ucl.ac.uk/bsm/dbbrowser/c32/contents.html">A taste of bioinformatics</a> (University College London)<br />
<a href="http://bio5495.wustl.edu/">Introduction to Computational Molecular Biology</a> (Washington University in St. Louis)<br />
<a href="http://www.bioinformaticscourses.com/bioinform/index.html">Introduction to Bioinformatics</a> (UCSD Extension)<br />
<a href="http://www.cs.washington.edu/education/courses/527/01au/">Computational Biology</a> (University of Washington)<br />
<a href="http://theory.lcs.mit.edu/~bab/class/01-18.417-home.html">Introduction to Computational Molecular Biology</a> (MIT)<br />
<a href="http://bozeman.mbt.washington.edu/compbio/mbt599/">Introduction to Computational Molecular Biology</a> (University of Washington)</p>

<ul>
<li><strong>Lectures from University of Washington in Seattle US （recommended by Xiao-Yu Ni）</strong>
<br /></li>
</ul>

<p><a href="http://www.cs.washington.edu/education/courses/cse427/12wi/calendar/lecturelist.html">No. 1 lecture Bioinformatics and Biostatistics</a><br />
<a href="http://www.cs.washington.edu/education/courses/cse527/">No.2 Biostatistics</a><br />
<a href="http://www.cs.washington.edu/education/courses/cse586/10au/">No.3 Synthetic/Systems/Mathematical Biology</a></p>

<ul>
<li><strong>Algorithms  算法</strong>
<br /></li>
</ul>

<p><a href="http://www.cs.technion.ac.il/~cs236522/" title="Algorithms">Algorithms in Computational Biology</a> (Technion -Israel Institute of Technology)<br />
<a href="http://www.math.tau.ac.il/~rshamir/algmb.html">Algorithms for Molecular Biology</a> (School of Mathematical Sciences at Tel Aviv University)</p>

<ul>
<li><strong>Miscellaneous  其他</strong>
<br /></li>
</ul>

<p><a href="http://helix.biology.mcmaster.ca/chpt7.pdf">Elementary Sequence Analysis</a> (McMaster University)<br />
<a href="http://www.sbc.su.se/~per/molbioinfo2001/dynprog/dynamic.html">Dynamic Programming Tutorial</a> (By Eric C. Rouchka)<br />
<a href="http://www.rothamsted.bbsrc.ac.uk/notebook/courses/guide/">Beginner&rsquo;s Guide to Molecular Biology</a> (Rothamsted Research)<br />
<a href="http://www.genome.iastate.edu/edu/doe/index.html">A Primer on Molecular Genetics</a> (Iowa State University)<br />
<a href="http://lectures.molgen.mpg.de/online_lectures.html">Online Lectures on Bioinformatics</a> (Max Planck Institute for Molecular Genetics)<br />
<a href="http://www.ch.embnet.org/pages/courses2.html">EMBnet.org Courses</a> (EMBnet.org)<br />
<a href="http://cmgm.stanford.edu/biochem218/">Computational Molecular Biology</a> (Stanford University)<br />
<a href="http://www.genome.gov/COURSE2003/handouts.cfm">Current Topics in Genome Analysis</a> (handouts) (NIH)<br />
<a href="http://videocast.nih.gov/PastEvents.asp?c=7">Current Topics in Genomic Analysis (on-line video)</a> (NIH)<br />
<a href="http://www.sdsc.edu/pb/edu/pharm201/">Biological Data and Analysis Tools</a> (UCSD)<br />
<a href="http://www.bioinformaticscourses.com/ISB/index.html">Introduction to Structural Bioinformatics</a>(UCSD Extension)<br />
<a href="http://bioportal.weizmann.ac.il/course/prog/">Perl Programming Course for Bioinformatics and Internet</a> (Feinberg Graduate School of the Weizmann Institute of Science, Rehovot, Israel)<br />
<a href="http://bioportal.weizmann.ac.il/course/prog2/">Object-Oriented and Database Programming for Bioinformatics and Internet</a> (Feinberg Graduate School of the Weizmann Institute of Science, Rehovot, Israel)<br />
<a href="http://www.bioinformaticscourses.com/BIOL358/">Computer Skills For Biologists</a> (UCSD Extension)<br />
<a href="http://health.adelaide.edu.au/psychology/ccs/">Learning Statistics with R</a><br />
<a href="http://www.math.ufl.edu/help/matlab-tutorial/">Matlab Tutorial</a><br />
<a href="http://www.mathworks.com/academia/student_center/tutorials/intropage.html">Matlab Tutorial (Elementary)</a><br />
<a href="http://www.statsci.org/micrarra/">Microarray Data Analysis</a><br />
<a href="http://www.nanostring.com/products/gene_expression" title="NanoString Technologies">NanoString gene expression analysis</a><br />
<a href="http://lectures.gersteinlab.org/spring2002/show/pages/Slide02.htm">Computational Proteomics</a> (Genome-scale studies of protein function, structure, and evolution)<br />
<a href="http://www.medicalbillingandcodingonline.com/">Medical Billing and Coding Online</a></p></li>

<li><h3 id="background-information-news-背景信息与新闻">Background Information &amp; News  背景信息与新闻</h3>

<ul>
<li><a href="http://www.ncbi.nlm.nih.gov/education/" title="NCBI's Education">NCBI Education</a><br />

<ul>
<li>Probably the best starting point for anyone contemplating to switch to Bioinformatics</li>
</ul></li>
<li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=Books" title="e-Books">NCBI Bookshelf</a><br />

<ul>
<li>Includes a number of popular books in electronic format including Genomes by Brown and Human Molecular Genetics by Strachan.</li>
</ul></li>
<li><a href="http://www-test.ebi.ac.uk/2can/home.html" title="BioInfo - Toucan">EBI Education</a></li>
<li><a href="http://www.genome.gov/10000002">NIH Education</a></li>
<li><a href="http://www.ornl.gov/hgmis/">DOE Human Genome Project Information</a></li>
<li><a href="http://bioinformatics.org/">Bioinformatics.org</a><br />

<ul>
<li>The Open Lab</li>
</ul></li>
<li><a href="http://www.bio-computing.org/">Bio-computing.org</a><br />

<ul>
<li>Bioinformatics, Databases and Software for Medicine.</li>
</ul></li>
<li><a href="http://spot.colorado.edu/~dubin/bookmarks/b/1240.html">Online Science and Maths textbooks</a></li>
<li><a href="http://web.indstate.edu/thcme/mwking/subjects.html">Biochemistry online textbook</a></li>
<li><a href="http://prowl.rockefeller.edu/aainfo/contents.htm">Amino Acid Information</a></li>
<li><a href="http://www.worthington-biochem.com/index/manual.html">Worthington Enzyme Manual</a></li>
<li><a href="http://rambios.bioscinet.org/index-e.html">RAMBIOS</a></li>
<li><a href="http://www.wzw.tum.de/gene-quantification/">GENE QUANTIFICATION web page</a></li>
<li><a href="http://www.chem.qmw.ac.uk/iubmb/">International Union of Biochemistry and Molecular Biology nomenclature</a></li>
<li><a href="http://www.nlm.nih.gov/medlineplus/mplusdictionary.html">On-line Medical Dictionary Medline Plus</a></li>
<li><a href="http://www.onlinemedicalassistantprograms.com/">Online Medical Assistant Programs</a></li>
<li><a href="http://www.online-medical-dictionary.org/">Medical Dictionary Online</a></li>
<li><a href="http://www.futurebiojobs.com/">Futurebiojobs</a></li>
<li><a href="http://www.grantnet.com/">Funding Opportunities</a></li>
<li><a href="http://www.google.com/">Google</a><br />

<ul>
<li>Still THE BEST search engine on the web
<br />
<br />
<br /></li>
</ul></li>
</ul></li>

<li><h3 id="bioinformatics-conferences-国际会议">Bioinformatics Conferences  国际会议</h3>

<ul>
<li><a href="http://algo2018.hiit.fi/wabi/">Workshop on algorithms in bioinformatics (WABI)</a></li>
<li><a href="http://eccb.iscb.org/">European conference on computational biology (ECCB)</a></li>
<li><a href="http://www.recomb.org/home">International Conference on Research in Computational Molecular Biology (RECOMB)</a></li>
<li><a href="https://www.iscb.org/ismb2018">Annual international conference on intelligent systems for molecular biology (ISMB)</a></li>
<li><a href="http://bis.zju.edu.cn/binfo/links.htm#">IEEE Conference on computational systems bioinformatics (CSB)</a></li>
<li><a href="https://psb.stanford.edu/">Pacific Symposium on Biocomputing (PSB)</a></li>
<li><a href="http://icbcb.org/">International Conference on Bioinformatics and Computational Biology (ICBCB)</a></li>
<li><a href="http://www.icbbe.com/">International Conference on Biomedical Engineering and Bioinformatics (ICBEB)</a></li>
<li><a href="http://conf.bionet.nsc.ru/bgrssb2018/en/">International Conference on the Bioinformatics of Genome Regulation and Structure\Systems Biology (BGRS\SB)</a></li>
<li><a href="http://bis.zju.edu.cn/binfo/links.htm#">International Symposium on Integrative Bioinformatics</a></li>
<li><a href="http://bis.zju.edu.cn/binfo/links.htm#">International Workshop on Biological Processes and Petri Nets</a></li>
<li><a href="https://incob.apbionet.org/">International Conference on Bioinformatics (InCoB)</a></li>
<li><a href="http://bis.zju.edu.cn/binfo/links.htm#">International Young Scientists School“System biology and bioinformatics”</a></li>
<li><a href="https://www.asbmb.org/">The American Society for Biochemistry and Molecular Biology Annual Meeting (ASBMB)</a>
<br />
<br />
<br /></li>
</ul></li>

<li><h3 id="other-collections-其他资源">Other Collections  其他资源</h3>

<ul>
<li><a href="http://www.oxfordjournals.org/our_journals/nar/database/c/" title="NAR Databases">Nucleic Acids Research (NAR) database category list</a></li>
<li>[](<a href="http://www.oxfordjournals.org/our_journals/nar/webserver/c">http://www.oxfordjournals.org/our_journals/nar/webserver/c</a> &ldquo;NAR_2006_WebServer&rdquo;)<a href="http://www.oxfordjournals.org/our_journals/nar/database/c/" title="NAR Databases">Nucleic Acids Research (NAR) web server category list</a></li>
<li><a href="http://www.hsls.pitt.edu/obrc/" title="NAR_2006_WebServer">OBRC</a><br />

<ul>
<li>Online Bioinformatics Resources Collection</li>
</ul></li>
<li><a href="http://www.bio-computing.org/">Bioinformatics, Databases and Software for Medicine</a><br />

<ul>
<li>Covers recent literature, tutorials, links, bioinformatics database, jobs, and news, updated daily</li>
</ul></li>
<li><a href="http://www.liebertonline.com/">www.liebertonline.com</a><br />

<ul>
<li>critical mass of content exceeds 50,000 papers from more than sixty authoritative journals, all full-text searchable and linked to external bibliographic databases.</li>
</ul></li>
<li><a href="http://bioinformatics.oxfordjournals.org/">bioinformatics.oxfordjournals.org</a><br />

<ul>
<li>new developments in genome bioinformatics and computational biology</li>
</ul></li>
<li><a href="http://genamics.com/software/">SoftwareSeek</a></li>
<li><a href="https://github.com/danielecook/Awesome-Bioinformatics">Awesome Bioinformatics</a> 按照用途类别列举了生物信息学常用的一些工具和包，比较全，包括python，R及其他一些软件包</li>
<li><a href="http://www.genomeweb.com/">GenomeWeb</a></li>
<li><a href="http://www.expasy.org/links.html">Amos&rsquo; links</a></li>
<li><a href="http://www.expasy.ch/tools/">ExPASy Proteomics tools</a></li>
<li><a href="http://www.up.univ-mrs.fr/~wabim/english/logligne.html">Atelier Bioinformatique</a></li>
<li><a href="http://workbench.sdsc.edu/">Biology WorkBench</a></li>
<li><a href="http://www.bioinformatics.org/">MolBiol.Net</a></li>
<li><a href="http://bmerc-www.bu.edu/">BMERC</a></li>
<li><a href="http://www.nature.com/nrg/series/computational/index.html">Computational Tools</a> (Nature Reviews Genetics)</li>
<li><a href="http://biothink.com.au/">BioThink</a>: Bioinformatics Services, Scientific Editing, Synthetic Biology</li>
<li><a href="http://anil.cchmc.org/sms/index.html">SMS</a><br />

<ul>
<li>Sequence Manipulation Suite
<br />
<br />
<br /></li>
</ul></li>
</ul></li>

<li><h3 id="bioinformatics-journals-相关期刊">Bioinformatics Journals  相关期刊</h3>

<ul>
<li><a href="http://www.hindawi.com/journals/abi/">Advances in Bioinformatics</a></li>
<li><a href="http://www.almob.org/">Algorithms in Molecular Biology</a></li>
<li><a href="http://www.sciencedirect.com/science/journal/00043702">Artificial Intelligence</a></li>
<li><a href="http://bioinformatics.oxfordjournals.org/">Bioinformatics</a></li>
<li><a href="http://www.bioinformation.net/">Bioinformation</a></li>
<li><a href="http://www.bmjournal.in/">Biomirror Journal</a></li>
<li><a href="http://www.biomedcentral.com/bmcbioinformatics">BMC Bioinformatics</a></li>
<li><a href="http://www.biomedcentral.com/bmcgenomics/">BMC Genomics</a></li>
<li><a href="http://bib.oxfordjournals.org/">Briefings in Bioinformatics</a></li>
<li><a href="http://bfgp.oxfordjournals.org/">Briefings in Functional Genomics and Proteomics</a></li>
<li><a href="http://link.springer.com/journal/11538" title="Bulletin of Mathematical Biology">Bulletin of Mathematical Biology</a></li>
<li><a href="http://www.la-press.com/cancer-informatics-journal-j10" title="Cancer Informatics">Cancer Informatics</a></li>
<li>_[](file:///C:/wiki/Computers_in_Biology_and<em>Medicine &ldquo;Computers in Biology and Medicine&rdquo;)</em><a href="http://www.cbi.or.jp/cbi/englishindex.html">Chem-Bio Informatics Journal</a></li>
<li><a href="http://www.sciencedirect.com/science/journal/14769271">Computational Biology and Chemistry</a></li>
<li><a href="http://www.journals.elsevier.com/computers-in-biology-and-medicine/" title="Computers in Biology and Medicine">Computers in Biology and Medicine</a></li>
<li><a href="http://www.benthamscience.com/cbio/">Current Bioinformatics</a></li>
<li><em>[](file:///C:/wiki/Database</em>(journal) &ldquo;Database (journal)&rdquo;)_<a href="http://database.oxfordjournals.org/">Database</a></li>
<li><a href="http://genomebiology.com/">Genome Biology</a></li>
<li><a href="http://www.genome.org/">Genome Research</a></li>
<li><a href="http://www.journals.elsevier.com/genomics-proteomics-and-bioinformatics/">Genomics, Proteomics &amp; Bioinformatics</a></li>
<li><a href="http://hmg.oxfordjournals.org/">Human Molecular Gentics</a></li>
<li><a href="http://www.computer.org/portal/web/tcbb/home">IEEE/ACM Transactions on Computational Biology and Bioinformatics</a></li>
<li><a href="http://www.bioinfo.de/isb/">In Silico Biology</a></li>
<li><a href="http://paginas.fe.up.pt/~ijcvb/">International Journal for Computational Vision and Biomechanics</a></li>
<li><a href="https://www.inderscience.com/browse/index.php?journalcode=ijbra">International Journal of Bioinformatics Research and Applications (IJBRA)</a></li>
<li><a href="http://www.inderscience.com/browse/index.php?journalID=189">International Journal of Data Mining and Bioinformatics (IJDMB)</a></li>
<li><a href="http://journals.worldscientific.com.sg/jbcb/jbcb.shtml">Journal of Bioinformatics and Computational Biology (JBCB)</a></li>
<li><a href="http://www.sciencedirect.com/science/journal/15320464">Journal of Biomedical Informatics</a></li>
<li><a href="http://www.springer.com/journal/11373">Journal of Biomedical Science</a></li>
<li><a href="http://www.liebertpub.com/publication.aspx?pub_id=31">Journal of Computational Biology</a></li>
<li><a href="http://www.ripublication.com/jcib.htm">Journal of Computational Intelligence in Bioinformatics (JCIB)</a></li>
<li><a href="http://journal.imbio.de/">Journal of Integrative Bioinformatics - JIB</a></li>
<li><a href="http://link.springer.com/journal/285" title="Journal of Mathematical Biology">Journal of Mathematical Biology</a></li>
<li><a href="http://www.journals.elsevier.com/journal-of-molecular-biology/">Journal of Molecular Biology</a></li>
<li>_[](file:///C:/wiki/Journal_of_Theoretical<em>Biology &ldquo;Journal of Theoretical Biology&rdquo;)</em><a href="http://www.sciencedirect.com/science/journal/10933263">Journal of Molecular Graphics and Modelling</a></li>
<li><a href="http://link.springer.com/journal/894">Journal of Molecular Modelling</a></li>
<li><a href="http://pubs.acs.org/journal/jprobs">Journal of Proteome Research</a></li>
<li><a href="http://www.springerlink.com/content/106605/">Journal of Structural and Functional Genomics</a></li>
<li><a href="http://www.journals.elsevier.com/journal-of-theoretical-biology/" title="Journal of Theoretical Biology">Journal of Theoretical Biology</a></li>
<li>_[](file:///C:/wiki/PLoS_Computational<em>Biology &ldquo;PLoS Computational Biology&rdquo;)</em><a href="http://www.springerlink.com/content/0885-6125">Machine Learning</a></li>
<li><a href="http://www.sciencedirect.com/science/journal/10967176">Metabolic Engineering</a></li>
<li><a href="http://www.nature.com/msb/index.html">Molecular Systems Biology</a></li>
<li><a href="http://www.nature.com/">Nature</a></li>
<li><a href="http://www.nature.com/nbt/index.html">Nature Biotechnology</a></li>
<li><a href="http://www.nature.com/ng/index.html">Nature Genetics</a></li>
<li><a href="http://www.nature.com/nsmb/index.html">Nature Structural Biology</a></li>
<li><a href="http://nar.oxfordjournals.org/">Nucleic Acids Research</a></li>
<li><a href="http://www.liebertonline.com/loi/omi">OMICS: A Journal of Integrative Biology</a></li>
<li><a href="http://www.plosbiology.org/">PLoS Biology</a></li>
<li><a href="http://compbiol.plosjournals.org/">PLoS Computational Biology - A Peer-Reviewed Open-Access Journal</a></li>
<li><a href="http://www.plosgenetics.org/">PLoS Genetics</a></li>
<li><a href="http://www.proteinscience.org/">Protein Science</a></li>
<li><a href="http://www3.interscience.wiley.com/cgi-bin/jhome/36176">Proteins: Structure, Function, Bioinformatics</a></li>
<li><a href="http://www3.interscience.wiley.com/cgi-bin/jhome/76510741">Proteomics</a></li>
<li><a href="http://www.sciencemag.org/">Science</a></li>
<li><a href="http://www.degruyter.com/view/j/sagmb" title="Statistical Applications in Genetics and Molecular Biology">Statistical Applications in Genetics and Molecular Biology</a></li>
<li>[](<a href="http://www.almob.org/)[Structure](http://www.structure.org/">http://www.almob.org/)[Structure](http://www.structure.org/</a>)</li>
<li><a href="http://www.nature.com/emboj/index.html">The EMBO Journal</a></li>
<li><a href="http://www.computer.org/tcbb">Transactions on Computational Biology and Bioinformatics</a></li>
</ul></li>
</ul>

		</div>
		<footer class="post__footer">
			
<div class="post__tags tags clearfix">
	<svg class="tags__badge icon icon-tag" width="16" height="16" viewBox="0 0 32 32"><path d="M32 19c0 1-1 2-1 2L21 31s-1 1-2 1-2-1-2-1L2 16c-1-1-1.4-2-1.4-2S0 12.5 0 11V3C0 1.5.8.8.8.8S1.5 0 3 0h8c1.5 0 3 .6 3 .6S15 1 16 2l15 15s1 1 1 2zM7 10a3 3 0 1 0 0-6 3 3 0 0 0 0 6z"/></svg>
	<ul class="tags__list">
		<li class="tags__item">
			<a class="tags__link btn" href="/tags/bi/" rel="tag">Bi</a>
		</li>
		<li class="tags__item">
			<a class="tags__link btn" href="/tags/linux/" rel="tag">Linux</a>
		</li>
	</ul>
</div>
		</footer>
	</article>
</main>

<div class="authorbox clearfix">
	<figure class="authorbox__avatar">
		<img alt="Ricky Wang avatar" src="/img/avatar.png" class="avatar" height="90" width="90">
	</figure>
	<div class="authorbox__header">
		<span class="authorbox__name">About Ricky Wang</span>
	</div>
	<div class="authorbox__description">
		Ricky is Scientist
	</div>
</div>



			</div>
			<aside class="sidebar"><div class="widget-search widget">
	<form class="widget-search__form" role="search" method="get" action="https://google.com/search">
		<label>
			<input class="widget-search__field" type="search" placeholder="SEARCH…" value="" name="q" aria-label="SEARCH…">
		</label>
		<input class="widget-search__submit" type="submit" value="Search">
		<input type="hidden" name="sitesearch" value="/">
	</form>
</div>
<div class="widget-taglist widget">
	<h4 class="widget__title">Tags</h4>
	<div class="widget__content">
		<a class="widget-taglist__link widget__link btn" href="/tags/another-test/" title="Another Test">Another Test</a>
		<a class="widget-taglist__link widget__link btn" href="/tags/bi/" title="Bi">Bi</a>
		<a class="widget-taglist__link widget__link btn" href="/tags/linux/" title="Linux">Linux</a>
		<a class="widget-taglist__link widget__link btn" href="/tags/test/" title="Test">Test</a>
	</div>
</div>
</aside>
		</div>
		<footer class="footer">
	<div class="container footer__container flex">
		
		<div class="footer__copyright">
			&copy; 2023 DNA ACADEMIC
			<span class="footer__copyright-credits">| All About DNA.</span>
		</div>
	</div>
</footer>
	</div>
<script async defer src="/js/menu.js"></script>
<script src="/js/custom.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.6/MathJax.js?config=TeX-AMS-MML_HTMLorMML" async></script>
</body>
</html>